# #MybMonday

MYB transcription factors are one of the largest gene family in plants and control many processes. This repository provides additional background to the #MYB_Monday tweets. #MybMonday is maintained by the [PuckerLab](https://www.tu-braunschweig.de/en/ifp/pbb) at TU Braunschweig.

## Introduction ##

The MYB transcription factors from one of the largest transcription factor families in plants. The characteristic feature of MYBs are repeat units which harbour conserved tryptophan (W) residues with a defined number of interleaved amino acid residues ([Dubos et al., 2010](https://doi.org/10.1016/j.tplants.2010.06.005)). These repeat units determine the names of MYBs. For example, the 'R2R3 MYBs' harbour two MYB repeat units: the R2 and the R3 repeat. Especially the R2R3 clade of the MYB family is expanded in plants. Members of this family are phylogenetically assigned to groups. It is expected that members of a certain group control a specific function or process. Therefore, it is possible to transfer the knowledge about MYBs studied in one plant species to the MYBs identified in a new species. Due to the large number of MYBs, they are involved in numerous processes.

Here is a list of the expected functions of [MYBs in banana (_Musa acuminata_)](https://doi.org/10.1371/journal.pone.0239275): (1) defense/stress response, (2) trichome branching and petal morphogenesis, (3) defense/stress response, (4) glucosinolates/camalexin, (5) suberin, (6) root development, (7) phenylpropanoid/lignin, (8) flower meristem identity, (10) abiotic stress response, flower morphogenesis, and stilbene biosynthesis, (11) photomorphogenesis, (12) anther and tapetum development, (14) axillary meristem and root growth, (15) SCW and lignin, (16) stemen/anther development, phenylpropanoid, SCW, (17) anther and trichome development, (18) SCW and lignin, (20) mucillage, lignin, and stomatal closure, (22) repressors of phenylpropanoid, sinapata, lignin, (23) general flavonoid biosynthesis and trichome development, (24) anthocyanins (monocots), (25) proanthocyanins, (27) flavonoid repressors, (28) trichome development, roothair development, floral organ development, (29) anthocyanins and betalains (dicots), (30) flavonols and phlobaphene, (31) cell cycle regulation, (33) phenylpropanoid and lignin, (34) stamen development, (35) defense and stress response, (36) leaf, shoot, germ morphogenesis, (37) anther development and stress response, (38) embryogenesis and seed maturation, (39) guard cell division and root gravitropism, (40) stress response and hormone signaling, (41) stress tolerance, and (42) cell wall, lignin, seed oil,axillary meristem. The phylogenetic relationship of these MYBs is displayed in Fig. 1.


<a href="https://doi.org/10.1371/journal.pone.0239275">
<img alt="Phylogenetic tree of banana MYBs" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/03/2020.02.03.932046/F2.large.jpg" width="75%" height="75%">
</a>

Fig. 1: Phylotenetic tree of the banana (_Musa acuminata_) MYBs ([Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275)).


In addition to the R2R3 MYBs, there are also [MYB3R (cell cycle control)](https://dev.biologists.org/content/134/6/1101), [MYB4R (SNAP complex)](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjJnMGJ6vruAhWLyaQKHc3rAysQFjABegQIBBAD&url=https%3A%2F%2Facademic.oup.com%2Fplphys%2Fadvance-article-pdf%2Fdoi%2F10.1093%2Fplphys%2Fkiaa067%2F35063342%2Fkiaa067.pdf&usg=AOvVaw04BwHFjy1QcAHg9-k5KeOO), and various others. However, these lineages have not expanded substantially in plants.

More information about MYBs involved in the regulation of the flavonoid biosynthesis is available through the **[FlavonoidFriday](https://github.com/bpucker/FlavonoidFriday)**.

Check out the collection of [genome-wide MYB investigations](https://github.com/bpucker/MybMonday/wiki/Collection-of-genome-wide-MYB-papers). Please get in touch to add entries to this collection.



## Tweets with additional background ##

### (1) MYBs control diverse functions in plants ###

> **_<a href="https://twitter.com/boas_pucker/status/1363737787782995970">TWEET</a>:_** MYBs are among the largest transcription factor families in plants. Members of the R2R3-MYB clade control various processes including ontogenesis, cell wall biosynthesis, stress response, and specialized metabolism. #MybMonday (src: https://bit.ly/2NkoaYZ)

<a href="https://twitter.com/boas_pucker/status/1363737787782995970">
<img alt="Phylogenetic tree of banana MYBs (Tweet #1)" src="https://pbs.twimg.com/media/EuzsUIJXcAEPIpy?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

(Figure source: [Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275))

This tweet serves as an introduction. Please read the introduction section on top of this page for additional information. The phylogenetic tree shows the diversity of different functions that are regulated by MYB transcription factors. It appears that there are cases of convergent evolution i.e. different clades of MYBs regulating the same process.


### (2) MYB repeat structure ###

> **_<a href="https://twitter.com/boas_pucker/status/1363858585768263686">TWEET</a>:_** Characteristic feature of MYBs are conserved tryptophan residues at defined positions in MYB repeat units. Names of MYBs are based on these MYB repeats e.g. 'R2R3-MYBs' harbour one R2 and one R3 repeat. #MybMonday (src: https://bit.ly/2ZBfWOD)

<a href="https://twitter.com/boas_pucker/status/1363858585768263686">
<img alt="MYB repeat structure (Tweet #2)" src="https://pbs.twimg.com/media/EuzuOVBWQAAfvEM?format=png&name=medium" width="50%" height="50%">
</a>


These characteristic repeat features can be harnessed for the identification of MYBs in transcriptome or genome assemblies. Regular expressions can be used to search for this amino acid pattern in a given set of peptide sequences. Many genome-wide studies of R2R3-MYBs use the presence of this conserved domain to select their candidates.


### (3) MYBs interact with bHLH and WD40 (MBW complex) ###

> **_<a href="https://twitter.com/boas_pucker/status/1366274503207227392">TWEET</a>:_** Some MYBs form complexes with bHLH and WD40 proteins. These complexes are named MBW after the 3 involved TFs. MBW controls the seed coat colour in Arabidopsis thaliana and other developmental processes. #MybMonday (src: https://bit.ly/2ZCoswD)

<a href="https://twitter.com/boas_pucker/status/1366274503207227392">
<img alt="MYBs interact with bHLH and WD40 (Tweet #3)" src="https://pbs.twimg.com/media/EvNNzvcXIAEq1tv?format=jpg&name=medium" width="30%" height="30%">
</a>


Different MYBs and bHLHs can be involved in the MBW complex formation. The integration of different TFs allows specificity for different target genes. This complex is not only required for the regulation of the proanthocyanidin biosynthesis, but also plays a role in the development of trichomes and root hairs.

### (4) MYBs are breeding targets ###

> **_<a href="https://twitter.com/boas_pucker/status/1368811218014629888">TWEET</a>:_** MYBs are the target of breeding projects to improve fruits, vegetables, and ornamental plants. MYB1/10/110a are responsible for the red coloration of apple skin, foliage & fruit-flesh, respectively, by activating a pigment biosynthesis  #MybMonday (src: https://bit.ly/37OrRgb)

<a href="https://twitter.com/boas_pucker/status/1368811218014629888">
<img alt="MYBs are breeding targets (Tweet #4)" src="https://pbs.twimg.com/media/EvNPR_CXMAEvIFp?format=jpg&name=medium" width="30%" height="30%">
</a>


As transcriptional regulators of entire biosnythesis pathways, MYBs are promising targets of breeding projects. The deactivation of a specific MYB and be used to shut down a pathway leading to undesired products. Additionally, the contruction of genetically modified plants, e.g. overexpressing certain MYBs, is an option to improve the crop quality by up-regulating desired pathways.


### (5) MYB associated with sweet cherry colour ###

> **_<a href="https://twitter.com/boas_pucker/status/1371347932851634177">TWEET</a>:_** PavMYB10.1 is associated with the fruit skin colour of sweet cherry. Three different alleles lead to yellow to red colouration in the varieties 'Big Dragon', 'Rainier', and 'Lapins'. #MybMonday (src: https://bit.ly/3swLEsx)

<a href="https://twitter.com/boas_pucker/status/1371347932851634177">
<img alt="MYB associated with sweet cherry colour (Tweet #5)" src="https://pbs.twimg.com/media/EvNSbz7XMAAZqcG?format=jpg&name=medium" width="30%" height="30%">
</a>


### (6) Red and white strawberries ###

> **_<a href="https://twitter.com/boas_pucker/status/1373884648485445633">TWEET</a>:_** The red colour of octoploid strawberry (Fragaria Ã— ananassa) is controlled by a MYB. A 8bp insertion in the CDS of FaMYB10 can change the colour from red to white through a single loss-of-function allele. #MybMonday (src: https://bit.ly/2ZUIGBV)

<a href="https://twitter.com/boas_pucker/status/1373884648485445633">
<img alt="Red and white strawberries (Tweet #6)" src="https://pbs.twimg.com/media/EvNUujRWgAIgh6r?format=jpg&name=medium" width="30%" height="30%">
</a>


This loss-of-function allele of MYB10-2 needs to be present in a homozygous state to cause the white phenotype as it is not dominant negative. Interesting about the MYB10-2 is the lack of introns wich suggest that it originates from a reverse transcription event. It will be exciting to completely resolve the story about the anthocyanins regulating MYBs in various strawberry strains. It is also a remarkable example of pathway inactivation through loss of a regulator. This is frequently observed in the flavonoid biosynthesis. One reason could be the involvement of structural genes in multiple branches of the pathway. The loss of the anthocyanin specific regulator only inactivaites one function, while loss of the structural gene would break multiple branches.


### (7) Colouration of kiwi fruit tissues ###

> **_<a href="https://twitter.com/boas_pucker/status/1376406263693783042">TWEET</a>:_** The red colour of kiwifruit petals is determined by MYB110a, but there might be more MYBs involved in the anthocyanin biosynthesis regulation in other tissues. A microsatellite marker linked to MYB110a revealed the location on chr10. #MybMonday (src: https://bit.ly/3aVRWvZ)

<a href="https://twitter.com/boas_pucker/status/1376406263693783042">
<img alt="Colouration of kiwi fruit tissues (Tweet #7)" src="https://pbs.twimg.com/media/EvNXel3XcAAW_75?format=jpg&name=large" width="50%" height="50%">
</a>


### (8) Red blushed skin of apricots ###

> **_<a href="https://twitter.com/boas_pucker/status/1378942978253807616">TWEET</a>:_** Consumers prefer apricots with red blushed skin. Expression of PaMYB10 correlates with expression of anthocyanin biosynthesis genes which are responsible for production of red pigments. PaMYB10 seems to be an important activator. #MybMonday (src: https://bit.ly/2ZXbWbo)

<a href="https://twitter.com/boas_pucker/status/1378942978253807616">
<img alt="Red blushed skin of apricots (Tweet #8)" src="https://pbs.twimg.com/media/EvNbhhvXYAE5-2R?format=png&name=900x900" width="30%" height="30%">
</a>


### (9) Betalain biosynthesis in sugar beet is controlled by BvMYB1 ###

> **_<a href="https://twitter.com/boas_pucker/status/1381479694483156992">TWEET</a>:_** BvMYB1 is responsible for the red colouration of sugar beets and beetroots. This MYB activates the betalain biosynthesis which produces red pigments. #MybMonday (src: https://bit.ly/3q0w2fa)

<a href="https://twitter.com/boas_pucker/status/1381479694483156992">
<img alt="Betalain biosynthesis in sugar beet is controlled by BvMYB1 (Tweet #9)" src="https://pbs.twimg.com/media/EvNeVhRXAAUsYb0?format=png&name=900x900" width="30%" height="30%">
</a>

Anthocyanin and betalains are (to some extent) functionally redundant. Not a single plant species was detected to show pigmentation by anthocyanins and betalains which lead to the mutual exclusion theory ([Timoneda et al., 2019](https://doi.org/10.1111/nph.15980)). BvMYB1 is an ortholog of the anthocyanin regulator PAP1. The co-option of this anthocyanin regulator by the betalain biosynthesis makes the complex pigment evolution in the Caryophyllales even more exciting. In contrast to PAP1, BvMYB1 does not require a bHLH and TTG1 cofactor to activate the betalain biosynthesis. The loss of an anthocyanin regulator to the betalain biosynthesis could be one explanation why anthocyanins have been replaced by betalains in sugar beet.


### (10) Flavonol regulator BvMYB12 ###

> **_<a href="https://twitter.com/boas_pucker/status/1384016409202679808">TWEET</a>:_** MYB functions are not restricted to fruit colours. BvMYB12 activates flavonol biosynthesis genes in sugar beet independent of bHLH/WD40. Flavonols protect plants against UV light and ROS, but also regulate developmental processes. #MybMonday (src: https://bit.ly/3dSMgVo)


<a href="https://twitter.com/boas_pucker/status/1384016409202679808">
<img alt="Flavonol regulator BvMYB12 (Tweet #10)" src="https://pbs.twimg.com/media/EvNku2gWYAEWAeN?format=jpg&name=medium" width="40%" height="40%">
</a>

(Figure source: [Stracke et al., 2014](https://doi.org/10.1186/s12870-014-0249-8))

While betalains replace anthocyanins in many Caryophyllales families ([Timoneda et al., 2019](https://doi.org/10.1111/nph.15980)), flavonols are still present in all Caryophyllales lineages. The flavonol biosynthesis is an evolutionary old pathway which appears to be conserved across all land plant lineages. The identificaiton of the AtMYB ortholog BvMYB12, which is also a flavonol biosynthesis regulator, indicates that the regulation of this pathway is also conserved over long phylogenetic distances.


### (11) Structure of WEREWOLF ###

> **_<a href="https://twitter.com/boas_pucker/status/1386553123402043392">TWEET</a>:_** Third alpha-helices of R2 and R3 repeats of WEREWOLF fit in the major groove of the DNA and recognize AACNGC. L59 of WER is probably sensing the methylation status of the binding site. #MybMonday (details: https://bit.ly/3nmpR5n)


<a href="https://twitter.com/boas_pucker/status/1386553123402043392">
<img alt="Structure of WEREWOLF (Tweet #11)" src="https://pbs.twimg.com/media/Ez2Ah4qXsAQKyCW?format=jpg&name=medium" width="40%" height="40%">
</a>

WEREWOLF (WER) is an important regulator in the plant ontogenesis [Lee & Schiefelbein, 1999](https://doi.org/10.1016/S0092-8674(00)81536-6). Generally, cis-elements of R2R3-MYBs can be assigned to the (C/T)AACNG or the ACC(A/T)A(A/C) group. A recent study by [Wang et al., 2019](https://academic.oup.com/nar/article/48/1/460/5626523) investigated the structure of WER and the interaction with cis-elements: The third alpha-helices of R2 and R3 repeats of WEREWOLF fit in the major groove of the DNA and recognize AACNGC. K55, N106, N109, and N110 are required for WER function. The L59 of WER is probably involved in sensig DNA methylation at 5mC. This 5mC methylation or a 6mA methylation in the AAC element might be able to prevent the interaction of WER and the DNA. L59 is a substitution in WER compared to MsMyb, TvMyb1, and TvMyb3. The L59A mutant shows almost the same affinity to methylated and unmethylated binding sites.


### (12) MYB2 of Boehmeria nivea ###

> **_<a href="https://twitter.com/boas_pucker/status/1389089839534968835">TWEET</a>:_** The R1-MYB family member MYB2 of Boehmeria nivea (Ramie) appears to be a positive regulator of Cd tolerance and accumulation. #MybMonday (details: https://bit.ly/3evEiQH)

<a href="https://twitter.com/boas_pucker/status/1389089839534968835">
<img alt="MYB2 of <i>Boehmeria nivea</i> (Tweet #12)" src="https://pbs.twimg.com/media/Ez2HyD-XEAApfcv?format=png&name=900x900" width="40%" height="40%">
</a>


A recent publication by [Zhu et al., 2020](https://doi.org/10.1371/journal.pone.0233375) describes MYB2 of Boehmeria nivea (Ramie) as regulator of Cadmium (Cd) tolerance and accumulation. BnMYB2 is an ortholog of GmMYB176, StMYB1R-1, AtMYBS3, OsMYBS1, and OsMYBS2. This clade is characterized by a conserved SHAQKYF motif. Heterologous expression of BnMYB2 in A. thaliana seem to increase the Cd tolerance and accumulation significantly. Would the expression of other MYBs e.g. AtMYBS3 result in a similar effect?


### (13) Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae ###

> **_<a href="https://twitter.com/boas_pucker/status/1391634107545829376">TWEET</a>:_** Subgroup 6 of the R2R3-MYBs controls the flavonoid biosynthesis which is responsible for the colouration of flowers. These MYBs contribute to the diversity of floral colour in Aristolochiaceae. #MybMonday (details:https://bit.ly/3y2PhdA)


<a href="https://twitter.com/boas_pucker/status/1391634107545829376">
<img alt="Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae (Tweet #13)" src="https://pbs.twimg.com/media/E1APsxnX0AQc5Af?format=jpg&name=medium" width="40%" height="40%">
</a>


Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae
[Munoz-Gomez et al.](https://doi.org/10.3389/fpls.2021.633227) report that the Aristolochiaceae are characterized by a huge diversity of flower coloration and forms. While the subgroup 6 of the R2R3-MYBs is generally responsible for regulation of the flavonoid biosynthesis (which includes colourful anthocyanins), there are several lineage-specific MYB gene duplications which cause alterations in the activity of regulated biosynthesis pathways. These TF differences result ultimately in the diversity of flower colors through different activation of pigment pathways. The authors investigated the expression of flavonoid biosynthesis genes in multiple species. No F3'5'H was detected in the transcriptome analyses which is in line with the absence of blue pigmented flowers. DFR and ANS were not detected in some of the datasets which would indicate that no anthocyanins are produced in the investigated tissues.


### (14) Conserved motifs in the C-terminal part of anthocyanin activators ###

> **_<a href="https://twitter.com/boas_pucker/status/1394178368149458946">TWEET</a>:_** Activators of the anthocyanin biosynthesis belong to subgroup 6 of the R2R3-MYB family. In addition to the R2R3 domain, there are three short motifs conserved among anthocyanin regulators #MybMonday (details: https://bit.ly/3eTQH28)

<a href="https://twitter.com/boas_pucker/status/1394178368149458946">
<img alt="Conserved motifs in the C-terminal part of anthocyanin activators (Tweet #14)" src="https://pbs.twimg.com/media/E1karEGWEAE1Cu-?format=jpg&name=large" width="40%" height="40%">
</a>


[Rodrigues et al., 2021](https://www.nature.com/articles/s41438-021-00514-1) identified functional variation in the C-terminal region of anthocyanin-activating MYBs. Molecular recognition features (MoRFs) are located in regions of intrinsic disorder of transcription factors. They are often short linear motifs (SLiMs) of 3-11 amino acid residues with evolutionary variability, but some consereved residues which determine specificity in interactions [Stracke et al., 2001](https://pubmed.ncbi.nlm.nih.gov/11597504/). Flexibility in these regions is considered an important factor in the evolution of signalling pathways by interacting co-factors. Multiple subgroups of the MYBs interact with bHLH transcription factors, but the involved motifs evolved convergently in the C-terminal region (subgroup 4 and subgroup 12) or in the N-terminal region (subgroups 5, 6, 15, 16). Anthocyanin regulating MYBs are located in subgroup 6. The C-terminal motif in this subgroup overlaps with the TAS4-siRNA81(-) target sequence. The PA and anthocyanin regulation subgroups 5 und 6 appear to be derived from the flavonoid regulator subgroup 7. However, early branching members in the subgroup 6 are not able to activate anthocyanins or lead only to a weak activation. Functionally characterized subgroup 6 members show conservation of three short motifs in their C-terminal region.


### (15) MYB involved in sex determination ###

> **_<a href="https://twitter.com/boas_pucker/status/1396707538373156864">TWEET</a>:_** The R2R3-MYB gene DEFECTIVE IN TAPETAL DEVELOPMENT AND FUNCTION (TDF1) is one of two sex determination genes in Asparagus officinalis. #MybMonday (details: https://bit.ly/3yHB6Ll)

<a href="https://twitter.com/boas_pucker/status/1396707538373156864">
<img alt="MYB involved in sex determination (Tweet #15)" src="https://pbs.twimg.com/media/E2H9ewXWYAAH912?format=jpg&name=medium" width="30%" height="30%">
</a>


Dioecy has evolved hundreds of times independently from hermaphrody in plants. [Harkess et al., 2020](http://www.plantcell.org/content/32/6/1790) present a MYB that is involved in sex determination in _Asparagus officinalis_. The X and Y chromosomes of A. officinalis evolved from an ancestral autosome pair during the shift from hermaphroditism to dioecy. A loss of recombination in a Y chromosomal region with genes supressing the female organogenesis (_SOFF_) and promoting male function (_TDF1_) can be caused by hemizygosity. Two Y-speficific genes appear sufficient for sex determination. The analysis of different mutant lines revealed a 1Mbp region on the Y-chromosome that is responsible for the sex determination. One of the genes in this region is the R2R3-MYB transcription factor _DEFECTIVE IN TAPETAL DEVELOPMENT AND FUNCTION_ (_TDF1_). Interestingly, this gene is not sex linked in all dioceious Asparagus species which indicates that this sex determination mechanism evolved within the genus. Sex chromosome evolution often leads to the accumulation of male genes on the Y chromosome and female genes on the X chromsome. One example of an X-linked gene is _WIP2/NTT_ which is specificly expressed in ovules.


### (16) MYBC1 in kiwifruit ###

> **_<a href="https://twitter.com/boas_pucker/status/1399261076877885443">TWEET</a>:_** Kiwifruit MYBC1 is an activator of the flavonoid biosynthesis genes F3'H and F3'5'H. Up-regulation F3'H/F3'5'H is discussed in context of increased proanthocyanidin content #MybMonday (details: https://bit.ly/3vBUJlH)


<a href="https://twitter.com/boas_pucker/status/1399261076877885443">
<img alt="MYBC1 in kiwifruit (Tweet #16)" src="https://pbs.twimg.com/media/E2ssKpOXoAIEqiW?format=jpg&name=large" width="30%" height="30%">
</a>



[Peng et al., 2020](https://www.nature.com/articles/s41598-020-70977-0) present MYBC1 and WRKY44 as activators of anthocyanin and proanthocyanidin biosynthesis. Both transcription factors appear to activate F3'H and F3'5'H as well as the proanthocyanidin biosnythesis genes. The authors present that an increased hydroxylation (via F3'H and F3'5'H) increases the flux into the proanthocyanidin biosynthesis which competes with the anthocyanin biosynthesis for substrate. Member of the MYB subgroup 6 are usually activating the anthocyanin biosynthesis. However, MYBC1 belongs to the subgroup 5. Peng et al. used RNA-Seq of differently pigmented kiwifruit cultivars to identify the underlying genes. Transient overexpression of two candidate genes, MYBC1 and WRKY44, in tobacco caused accumulation of the anthocyanin in patches. MYBC1 overexpression leads to cyanidin-based and delphinidin-based anthocyanin, while WRKY44 overexpression only caused the accumulation of cyanidin-based anthocyanins.
Overexpression in calli revealed differences between MYB110 (anthocyanin actiator) and MYBC1/WRKY44. While MYB110 activates the F3GT which is required for anthocyanin biosynthesis, MYBC1/WRKY44 actiate the F3'H/F3'5'H and proanthocyanidin biosynthesis genes. Since the MYBC1/WRKY44 overexpression results in higher proanthocyanidin content, the authors speculate that both TFs might be PA regulators rather than general activators of the flavonoid biosynthesis.



### (17) Evolution of the R2R3-MYBs in plants ###

> **_<a href="https://twitter.com/boas_pucker/status/1401811164737150977">TWEET</a>:_** Ancestral land plants already had 10 subfamilies of R2R3-MYBs. Orthologs of 7 subfamilies were detected in chlorophytes and charophycean algae. Subfamily VIII expanded in land plants. #MybMonday (details: https://bit.ly/3irG1tV)

<a href="https://twitter.com/boas_pucker/status/1401811164737150977">
<img alt="Evolution of the R2R3-MYBs in plants (Tweet #17)" src="https://pbs.twimg.com/media/E3Q59hoWEAA0RIj?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

A phylogenetic anaylysis by [Jiang & Rao, 2020](https://doi.org/10.1104/pp.19.01082) revealed 10 subfamilies of R2R3-MYBs in the ancestral land plants. Seven of these subfamilies have orthologs in the chlorophytes and charophycean algae. Different subfamilies of the MYB expanded in different plant lineages. Whole genome duplication alone cannot explain this pattern. Chlorophyta (<13) and streptophyte alga _Klebsormidium nitens_ (22) have small numbers of MYBs, while land plants have generally a larger number. About half of the land plant R2R3-MYBs fall into subfamily VIII, which is the main contributor to the expansion of R2R3-MYBs in land plants. This study supports the hypothesis that the R1R2R3-MYBs are not the result of R2R3-MYBs gaining an additional repeat.
The authors point out some issues when investigating the evolution of MYBs:
(1) R2R3-MYBs have been classified into 23-90 subgroups thus there is little consensus regarding their classification schemes. Therefore, Jiang & Rao propose a new classification scheme which better reflects the asymmetric evolution of R2R3-MYBs in land plants and is restricted to monophyletic groups.
(2) Phylogenetic anlysis are impacted by the quality of the annotations of studied species. The numbers of R2R3-MYBs might be underestimated in some species.


### (18) R2R3-MYBs regulate gucosinolate biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1404310128912785412">TWEET</a>:_** MYB28, MYB29, MYB34, MYB51, MYB76, and MYB122 control the glucosinolate biosynthesis. Sulfur deficiency reduces the gene expression of MYB29 and MYB76 through negative regulation by SLIM1. #MybMonday (details: https://bit.ly/2RWQNh6)

<a href="https://twitter.com/boas_pucker/status/1404310128912785412">
<img alt="R2R3-MYBs regulate gucosinolate biosynthesis (Tweet #18)" src="https://pbs.twimg.com/media/E30ZdesWEAMCm7a?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

Plants need a sophisticated regulatory system to balance the flux of sulfur into glucosinolate (GSL) biosynthesis and central metabolism. MYB34, MYB51, and MYB122 control the indolic GSL (IG) biosynthesis, while MYB28, MYB29, and MYB76 control the aliphatic GSL (AG) biosynthesis. These MYBs also enhance sulfur assimilation through activation of the respective genes. IG molecules contain 2 sulfur atoms in their backbone, while AG molecules contain 3 sulfur atoms in their backbone. [Frerigmann & Gigolashvili, 2014](https://doi.org/10.3389/fpls.2014.00626) studied the role of R2R3-MYBs under sulfur deficiency. Previous studies had identified that sulfur assimilation pathway genes are up-regulated under sulfur deficiency and that GSL biosynthesis is reduced. Catabolism of GSL is one solution how sulfur can be made available in the cell again.

Sulfur Limitation 1 (SLIM1) was identified as a negative transcriptional regulator of MYB29 and MYB76. SLIM1 reduces the activities of both promoters in a trans-activation assay. Sulfur deficiency resulted in a reduction of the AG content. The biosynthesis of AGs might require a tighter regulation than the IG biosnythesis, because the AG molecules contain one additional sulfur atom compared to IGs. However, a 'low GSL signal' seems to prevent the SLIM1-based repression of the MYB29 and MYB76 expression. This was investigated in mutant plants with a block in the GSL biosynthesis. The authors speculate that a complete down-regulation of the MYBs is not possible, because the six MYBs are activating sulfur assimilarion genes in addition to the GLS biosnythesis. There are multiple scenarios which could explain the observed results and further research on the topic is required.


### (19) VvMYB86 - activator of PA and repressor of anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1406846844878168064">TWEET</a>:_** Vitis vinifera MYB86 seems to be a positive regulator of the proanthocyanidin biosynthesis (LAR), but a negative regulator of the anthocyanin biosynthesis (ANS & UFGT) #MybMonday (details: https://bit.ly/3xuWods)


<a href="https://twitter.com/boas_pucker/status/1406846844878168064">
<img alt="VvMYB86 - activator of PA and repressor of anthocyanin biosynthesis (Tweet #19)" src="https://pbs.twimg.com/media/E4W9KcqXwAUq9KV?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

[Cheng et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2020.613677/full) characterized the Vitis vinifera MYB86 through in vivo and in vitro experiments. It turns out that this MYB is a positive regulator of LAR, which is involved in the anthocyanin biosynthesis. LAR and ANR are the two enzymes of the PA biosynthesis. While ANR determines the total amount of proanthocyanidins, LAR determines the degree of PA polymerisation. While upregulating LAR, this VvMYB86 represses ANS and UFGT, which are involved in anthocyanin formation. Most previously characterized V. vinifera MYBs are dedicated regulators of one of the pathways. However, MYB5 in V. vinifera and additional MYBs in other species have been identified as regulators of both pathways, but these are positive regulators of both pathways. Nevertheless, MYBs with opposite effects on both pathways have been described in other species before.
Sequence analysis of VvMYB86 revealed the presence of a bHLH interaction motif suggesting that this MYBs might be involved in a MBW complex. Protoplast transfections confirmed that MYB86 is able to activate LAR and to slightly repress ANS and UFGT promoters.


### (20) Anthocyanin activating AtMYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1409383560037892101">TWEET</a>:_** The three anthocyanin biosynthesis regulators PAP1 (MYB75), PAP2 (MYB90), and MYB113 are expressed in different developmental stages of Arabidopsis thaliana leaves #MybMonday (details: https://bit.ly/3jxuQRd)

<a href="https://twitter.com/boas_pucker/status/1409383560037892101">
<img alt="Anthocyanin activating AtMYBs (Tweet #20)" src="https://pbs.twimg.com/media/E47LvU_WEAcx8W8?format=jpg&name=large" width="30%" height="30%">
</a>


There are three anthocyanin activating MYBs in _Arabidopsis thaliana_: MYB75, MYB90, and MYB113. Trans-acting siRNA 4 (TAS4) and microRNA828 (miR828) are repressing the gene expression of these MYBs. [Koo & Poethig, 2021](https://doi.org/10.1186/s13765-020-00584-0) show that the expression of _PAP1_ (_MYB75_), _PAP2_ (_MYB90_), and _MYB113_ change over developmental stages of _A. thaliana_ leaves. TAS4 seems to be most active in young leaves. The anthocyanin content is equal in all leaves during the first 3-4 weeks, but there is a substantially higher anthocyanin concentration in apical leaves in older plants. An additional finding is that _PAP2_ might be responsible for the anthocyanin increase under nutrient-deficient conditions, because its gene expression was increased 1000-fold. Different transcript abundances of _PAP1_ and _PAP2_ could be determined by different affinities of TAS4 for both MYBs.



### (21) MYB111 controls flavonol diversity in sunflowers ###

> **_<a href="https://twitter.com/boas_pucker/status/1411935374494998533">TWEET</a>:_** Cis-regulatory variation in the MYB111 gene contributes to floral pigment diversity of sunflowers. Patterns of UV-absorbing flavonol glycosides are crucial to pollinator attraction and might contribute to temperature regulation #MybMonday (details: https://bit.ly/3jLVZQe)

<a href="https://twitter.com/boas_pucker/status/1411935374494998533">
<img alt="MYB111 controls flavonol diversity in sunflowers (Tweet #21)" src="https://pbs.twimg.com/media/E5fbXVbXMAAtzj9?format=png&name=900x900" width="30%" height="30%">
</a>

[Todesco et al., 2021](https://doi.org/10.1101/2021.06.26.449999) investigate the regulation of the floral pigment diversity of sunflowers. Floral pigmentation is an outstanding example of adapative variation. A pattern of UV absorbing pigments cannot be seen by the human eye, but probably is crucial for the pollinator attraction. A proportion of the sunflower accessions lacks such "nectar guides" due to completely UV-absorbing ligules. This suggest that the pollinator orientation is not one of the flavonol functions in these flowers. GWAS revealed a region on Chr15 as causal for this phenotypic difference between accessions. While this region does not contain a promising candidate gene, MYB111 is located close by. MYB111 appears responsible for the flavonol formation in petals of _A. thaliana_. It seems that MYB12 is not involved in the activation of flavonol accumulation in this tissue type. The explanation is a spatial separation of _MYB111_ and _MYB12_ expression with only _MYB111_ being expressed in these tissues. _MYB111_ shows a particularly high expression in developing ligules. Huge variation in the promoter region of _MYB111_ was observed between sunflower accessions. It appears that "nectar guides" are stronger in cultivated accessions compared to the ancestral state in the wild types, which probably leads to more pollinator visits. UV-absorbing compounds could lead to temperatue increase and make the flower more attractive to pollinators, but the effect is probably neglectable. Flavonols are also involved in drought response by reducing water loss. Since evaporation results in cooling, this could be another mechanism to regulate the flower temperature. 


### (22) Pepper MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1414472089126510593">TWEET</a>:_** A genome-wide analysis of R2R3-MYBs in Capsicum annuum revealed six candidates that might regulate the capsaicin and dihydrocapsaicin biosynthesis #MybMonday (details: https://bit.ly/36m7N3Z)

<a href="https://twitter.com/boas_pucker/status/1414472089126510593">
<img alt="Pepper MYBs (Tweet #22)" src="https://pbs.twimg.com/media/E506B0LWYAM3wfV?format=jpg&name=large" width="30%" height="30%">
</a>

[Wang et al., 2020](https://doi.org/10.3389/fgene.2020.598183) identified 108 R2R3-MYBs in Capsicum annuum. Pepper is well known for its piciness which is due to capsaicin. The authors expected a connection between the MYBs and the capsaicin biosynthesis. Gene structure, protein domain structure, and phylogenetic relationships of the MYBs were summarized in one figure. Individual MYBs were identified as co-expressed with single genes of the capsaicinoid biosynthesis, but it appears that there is no general regulator. CaMYB108 was previously identified as a capsaicin biosynthesis regulator. There might be other/additional transcriptional regulators which were not identified in this study.


### (23) _Morella rubra_ MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1417008803967426561">TWEET</a>:_** _Morella rubra_ has 174 MYBs. Overexpression of MrMYB12 causes an increase in flavonol biosynthesis gene expression (_CHS_, _F3H_, _FLS_) and results in lower flux into anthocyanin biosynthesis #MybMonday (details: https://bit.ly/3xLwrae)

<a href="https://twitter.com/boas_pucker/status/1417008803967426561">
<img alt="_Morella rubra_ MYBs (Tweet #23)" src="https://pbs.twimg.com/media/E6Z3piNXIAMiulT?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


[Cao et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2021.691384/full) describe the genome-wide investigation of the MYB gene family in Chinese bayberry (_Morella rubra_). _M. rubra_ is a subtropical fruit tree native to China. The 174 MYBs include 122 R3R3-MYBs, 43 1R-MYBs, two 3R-MYBs, one 4R-MYB, and six atypical MYBs. Tandem duplications and larger segmental duplications contributed to the total number of observed MYBs. _M. rubra_ is known for the production of various flavonoids. Therefore, the MrMYBs involved in the flavonoid biosynthesis were investigated in detail. MrMYB12 was identified as a flavonol regulator that can activate the gene expression of _CHS_, _F3H_, and _FLS_. MrMYB39 and MrMYB58a can activate the LAR promoter indicating that they are proanthocyanin regulators.



### (24) _Mimulus_ MYB5a ###

> **_<a href="https://twitter.com/boas_pucker/status/1419530420115365888">TWEET</a>:_** MYB5a is a key regulator of the anthocyanin biosynthesis in the genus Mimulus and contributes to the flower coloration diversity through a reaction-diffusion network #MybMonday (details: https://bit.ly/3iNIAoB)

<a href="https://twitter.com/boas_pucker/status/1419530420115365888">
<img alt=" _Mimulus_ MYB5a (Tweet #24)" src="https://pbs.twimg.com/media/E7LPz2HXIAIXBs3?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


[Zheng et al., 2021](https://doi.org/10.1093/genetics/iyaa036) investigated the coloration of flowers in the genus _Mimulus_. _MYB5a_/_NEGAN_ was previously identified as a crucial factor determining the visual diversity. _NEGAN_/_MYB5_ is part of a reaction-diffusion system in _M. lewisii_ and _M. guttatus_ that leads to spots of red anthocyanin pigmentation in the nectar guide region [Ding et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32155414/). Genetic mapping revealed _MYB4_ and _MYB5a_ as candidates in a QTL for anthocyanin pigmentation. Subsequent gene expression analysis showed that _MYB5a_ is the most likely candidate due to a specific expression in the petals. The yellow flowering _M. luteus luteus_ and the anthocyanin-pigmented _M. luteus variegatus_ differ by an InDel in the third exon of _MYB5a_ which excludes the subgroup 6 motif from the _M. luteus variegatus_ MYB. However, the third and the fourth exon are very similar (duplication?) suggesting that the fourth exon contributes the subgroup 6 motif. It appears that there are two different splice variants of the _MYB5_: exon1-2-4 in the anthocyanin-pigmented tissues and exon1-2-3 in other tissues. RNAi knockdown of _MYB5a_ was used to demonstrate the involvement in anthocyanin biosynthesis regulation. _MYB5a_ is an activator of its own inhibitor _RTO_.


### (25) R2R3-MYBs in tea plant ###

> **_<a href="https://twitter.com/boas_pucker/status/1422070909108637696">TWEET</a>:_** Genome-wide investigation of R2R3-MYBs in Camellia sinensis revealed candidates for the regulation of galloylated catechin biosynthesis #MybMonday (details: https://bit.ly/3ykOWTd)

<a href="https://twitter.com/boas_pucker/status/1422070909108637696">
<img alt="R2R3-MYBs in tea plant (Tweet #25)" src="https://pbs.twimg.com/media/E7wycUrXEAcr9Zy?format=jpg&name=medium" width="30%" height="30%">
</a>

(This plant species might be misclassified and could be _C. sasanqua_ or a related hybrid - thanks for a comment on twitter!)
<br>

[Li et al., 2021](https://doi.org/10.1101/2021.07.21.453189) analazyed the R2R3-MYBs in _Camelia sinensis_ (tea plant) to find the regulator of the galloylated catechin biosynthesis. Catechins accound for 70% of the polyphenols in _C. sinensis_ and are the major health promoting factor. Most abundant are galloylated catechins (ECG and EGCG). A total of 118 R2R3-MYBs were discovered in _C. sinensis_ and presented in a phylogenetic tree with the _A. thaliana_ MYBs (Dubos et al., 2010). No tea plant MYB falls in the S12 group comprized of AtMYB28, AtMYB29, AtMYB76, AtMYB34, and AtMYB51. These are Brassicaceae-specific MYBs that regulate glucosinolate biosynthesis (Gigolashvili et al., 2007; Matus et al., 2008). Three MYB subgroups without _A. thaliana_ members were identified. The authors propose that these evolved in tea plant, but the presence of _Vitis vinifera_ MYBs in these clades suggest that they have been lost in _A. thaliana_ instead. The authors decided to focus on subgroup 5 due to expression in apical buds and young leaves where high catechin concentrations were detected. _CsMYB30_, _CsMYB34_, _CsMYB37_ and _CsMYB42_ revealed a strong co-expression with the catechin biosynthesis genes (_CsMYB32_ instead of _CsMYB42_ was observed in a validation experiment). 


### (26) Mutation in R2R3-MYB _GmTDF1-1_ triggers male sterility in soybean ###

> **_<a href="https://twitter.com/boas_pucker/status/1424626503233404934">TWEET</a>:_** Disruption of the R2R3-MYB GmTDF1-1 by a point mutation triggers male sterility in soybean. The very similar GmTDF1-2 is too weakly expressed to compensate this loss of GmTDF1-1 activity #MybMonday (details: https://bit.ly/3xw7ej1)

<a href="https://twitter.com/boas_pucker/status/1424626503233404934">
<img alt="Mutation in R2R3-MYB _GmTDF1-1_ triggers male sterility in soybean (Tweet #26)" src="https://pbs.twimg.com/media/E8U99J6WEAIg7qe?format=jpg&name=medium" width="30%" height="30%">
</a>

[Yu et al, 2021](https://link.springer.com/article/10.1007/s00122-021-03920-0) report that the R2R3-MYB transcription factor GmTDF1-1 is crucial for the anther development in soybean. A loss of this gene function by a point mutation resulted in male sterility. There is a very similar _TDF1-2_ gene, but insufficient expression prevents a compensation of the _TDF1-1_ function loss. Male sterility is an important trait in soybean, because it facilitates hybrid breeding.



### (27) Evolution of the 3R-MYB gene family ###

> **_<a href="https://twitter.com/boas_pucker/status/1427148120580706305">TWEET</a>:_** 3R-MYBs (R1R2R3-MYBs) are involved in the cell cycle and in abiotic stress response. It might be an adaptation to a sessile lifestyle. #MybMonday (details: https://bit.ly/3yF4eCs)

<a href="https://twitter.com/boas_pucker/status/1427148120580706305">
<img alt="Evolution of the 3R-MYB gene family (Tweet #27)" src="https://pbs.twimg.com/media/E8lCCjTWQAE78Bl?format=png&name=medium" width="30%" height="30%">
</a>


[Feng et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28444194/) investigated the evolution of the 3R-MYB gene family. Since most plant MYBs belong to the R2R3-MYB gene family, the other families are often neglected. It is important to note that this classification based on the presence of repeats might not reflect the phylogeny of MYBs. When investigating the function of MYBs, it should be better to focus on their phylogeny instead on the presence of repeats. 3R-MYBs are divided into three groups: A, B, and C. Synteny analysis indicates that the firsts 3R-MYB gene was amplified through segmental duplications in Amborella. Gene duplications within these lineages after the separation of monocots and dicots are the important factor that explains the large diversity seen today. The general gene structure of 3R-MYBs changed during evolution through the acquisition of additional introns.  What could have been the evolutionary advantage? There are many cases of alternative splicing, but only a small proportion seems to change the final protein. Most events appear to be unproductive splicing or are restricted to the UTR. These events might be an additional layer of gene expression regulation. Alternative splicing in the CDS could also cause the loss of a repressor motif leading to a hyperactive protein. Helix2 and helix3 encoded by a MYB repeat are binding DNA and thus more conserved. Especially helix3 does not show much variation, because this helix forms the direct contact to the DNA.



### (28) MYB quiz ###

> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** How much do you know about MYBs? This quiz allows you to check and learn more about the functions of A. thaliana MYBs: https://lnk.tu-bs.de/tLugzE. Please post results below #MybMonday (details: https://bit.ly/3B2uvve)


<a href="https://lnk.tu-bs.de/tLugzE">
<img alt="MYB quiz (Tweet #28)" src="https://raw.githubusercontent.com/bpucker/figures/main/MYBquiz.png" width="50%" height="50%">
</a>


You can find the [MYB quiz](https://lnk.tu-bs.de/tLugzE) on Google colab (a Python script). Unfortunately, you need to log in with a google account in order to run the script. The script is started by pressing the play button at the top left. Next, you should get text output that is asking about the function of randomly selected _Arabidopsis thaliana_ MYBs. Everything is also explained in the documentation of the script in colab.



### (29) Genome-wide analysis of MYBs in banana ###

> **_<a href="https://twitter.com/boas_pucker/status/1432221545871523842">TWEET</a>:_** Genome-wide analysis of MYBs in banana (Musa acuminata) revealed the large size of this gene family including 285 R2R3-MYBs #MybMonday (details: https://bit.ly/3jplE0D & https://bit.ly/3kAW0Fx)


<a href="https://twitter.com/boas_pucker/status/1432221545871523842">
<img alt="Genome-wide analysis of MYBs in banana (Tweet #29)" src="https://pbs.twimg.com/media/E-BCE_RXEAElM5Y?format=jpg&name=small" width="50%" height="50%">
</a>

An investigation by [Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275) identified 285 R2R3-MYBs as well as several MYBs of other categories. These MYB genes were studied with respect to genomic position, gene structure and specific gene expressions patterns. Gene functions were predicted based on knowledge about previously characterized orthologs.



### (30) Anthocyanin repressor in pepper ###

> **_<a href="https://twitter.com/boas_pucker/status/1434765810589855748">TWEET</a>:_** Analysis of the R2R3-MYBs in pepper reveals CaMYB101 as candidate for the anthocyanin repression during fruit ripening #MybMonday (details:  https://bit.ly/3h86yee)

<a href="https://twitter.com/boas_pucker/status/1434765810589855748">
<img alt="Anthocyanin repressor in pepper (Tweet #30)" src="https://pbs.twimg.com/media/E-jGy8bX0Ac4wL5?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

[Liu et al., 2021](https://doi.org/10.1101/2021.08.24.457473) investigated R2R3-MYBs in pepper and present CaMYB101 as anthocyanin repressor. Some pepper cultivars accumulate anthocyanins in fruits during development and loose these pigments during fruit development. Since MYBs are important regulators of the anthocyanin biosynthesis, it is likely that they are involved in this process. This analysis is based on newly identified MYBs in _Capsicum baccatum_ and _C. chinense_ with previously identified R2R3-MYBs of _C. annuum_ from [Wang et al., 2020](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7779616/). CaMYB101 was identified based on orthology to PhMYB27 and SlMYBL2 which have been previously been reported as anthocyanin repressors. Gene expression analysis and VIGS experiments support this prediction.


### (31) R2R3-MYBs in sugar beet ###

> **_<a href="https://twitter.com/boas_pucker/status/1437302525179285509">TWEET</a>:_** Numbers of R2R3-MYBs differ substantially between plant species. Sugar beet is at the lower end of the range with only 70 R2R3-MYBs #MybMonday (details: https://bit.ly/3E8TYFv)

<a href="https://twitter.com/boas_pucker/status/1437302525179285509">
<img alt="R2R3-MYBs in sugar beet (Tweet #31)" src="https://pbs.twimg.com/media/E_GrzWRXEAAmk1z?format=jpg&name=large" width="50%" height="50%">
</a>

Sugar beet (_Beta vulgaris_) belongs to the large flowering plant order Caryophyllales. There are many reasons to study the MYB gene family in this species. 
Sugar beet is an important crop species in the northern hemisphere accounting for 30% of sugar production. The sugar beet genome sequence was also for a long time the only high quality genomic resource that represented the Caryophyllales. Particularly interesting is also the presence of betalains in this species while anthocyanins are absent (see [Timoneda et al., 2019](https://doi.org/10.1111/nph.15980) for a review). Since many MYBs are involved in the regulation of the anthocyanin/flavonoid biosynthesis, differences to other plants can be expected. The number of 70 R2R3-MYBs is small compared to other plants [(Stracke et al., 2014)](https://doi.org/10.1186/s12870-014-0249-8). For example, _Arabidopsis thaliana_ almost twice as many R2R3-MYBs despite the much smaller genome size. One potential explanation is the lack of recent gene duplications. For example, the BvMYB12 investigated by [Stracke et al., 2014](https://doi.org/10.1186/s12870-014-0249-8) does not have two recent duplications (MYB11, MYB111) as seen in _A. thaliana_.


### (32) R2R3-MYBs in banana ###

> **_<a href="https://twitter.com/boas_pucker/status/1439839239395500035">TWEET</a>:_** Banana (_Musa acuminata_) has one of the larger MYB families with 285 R2R3-MYBs in the A genome. Gene and segmental duplications are contributing factors #MybMonday (details: https://bit.ly/3tCDfW9)

<a href="https://twitter.com/boas_pucker/status/1439839239395500035">
<img alt="R2R3-MYBs in banana (Tweet #32)" src="https://pbs.twimg.com/media/E_GvdNFWUAclIQ_?format=jpg&name=medium" width="50%" height="50%">
</a>


While the number of 285 R2R3-MYBs in _Musa acuminata_ (banana) is impressive, not all of them are expressed ([Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275)). This already indicates that some of them might be duplicates that might be purged during evolution. There are often many copies in banana matching a single copy in other species. It is possible that sub- or neofunctionalization are ongoing, but in some cases these copies might be due to the lack of a strong selection against them. However, it is also important that this analysis was restricted to the A genome ([Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275)) thus there should be a similar number in the B genome.



### (33) Brown planthopper resistance ###

> **_<a href="https://twitter.com/boas_pucker/status/1442375955247423488">TWEET</a>:_** OsMYB30 confers brown planthopper (BPH) resistance to rice plants through activation of PALs in response to BPH feeding. This increases the content of salicylic acid and lignin #MybMonday (details: https://bit.ly/3kKgU6m)

<a href="https://twitter.com/boas_pucker/status/1442375955247423488">
<img alt="Brown planthopper resistance (Tweet #33)" src="https://pbs.twimg.com/media/FARHsujVEAA7l7a?format=jpg&name=large" width="50%" height="50%">
</a>

[He et al., 2019](https://www.pnas.org/content/117/1/271) identified a MYB transcription factor that confers brown planthopper (BPH) resistance to rice. BPH is one of the most destructive insects that damage rice by sucking sap from the phloem. It can also transmit viral diseases: rice grassy stunt and rugged stunt.
Several NBS-LRR resistance genes have been identified. It is known that lignin, salicylic acid, and polyphenols in general have a protective effect against insects. Enhancing the cell wall appears as an important mechanism to protect against insects. Gene expression between pools of BPH susceptible and resistant plants were compared and revealed PALs as a good candidate (OsPAL6,OsPAL8). Overexpression of these PALs confered resistance to a susceptible accession.
T-DNA knock-out of OsMYB30 resulted in a BPH susceptible line. The BPH feeding activates the expression of this MYB which activates the expression of a PAL (first step in general phenylpropanoid pathway).


### (34) DAP-Seq in grapevine ###

> **_<a href="https://twitter.com/boas_pucker/status/1444912670591754240">TWEET</a>:_** DAP-Seq in grapevine revealed the target genes of MYB13/14/15 (subgroup2): PAL, C4H, 4CL, shikimate pathway & stilbenoid pathway #MybMonday (details: https://bit.ly/2XV66cv)

<a href="https://twitter.com/boas_pucker/status/1444912670591754240">
<img alt="DAP-Seq in grapevine (Tweet #34)" src="https://pbs.twimg.com/media/FATlMIEXMAEuEbE?format=jpg&name=medium" width="30%" height="30%">
</a>

[Orduna et al., 2021](https://doi.org/10.1101/2020.12.31.424746) investigated the target genes of sugroup 2 R2R3-MYBs in grapevine via DNA affinity purification sequencing (DAP-Seq). The subgroup 2 comprises VvMYB14 and VvMYB15 as well as the newly characterized VvMYB13. These MYBs control the stilbene pathway which gives rise to resveratol. DAP-Seq revealed that these subgroup 2 MYBs bind to the promoters of many stilbene biosynthesis genes and also other genes involved in the specialized metabolism. There is also a substantial overlap between the target genes of MYB14 and MYB15. The integration of DAP-Seq results with gene centered expression networks allowed a refinement of the target gene set. Finally, overexpression of MYB15 supported this identified set of target genes. The analysis of VvMYB13 and the comparison to AtMYB13 suggests a diversification as only the AtMYB13 appears involved in lignin biosynthesis. Surprisingly, these grapevine MYBs also bind to genes that show an expression pattern opposite to stilbene biosynthesis. This observation might suggest a negative regulation through the subgroup 2 MYBs. Cell culture experiments suggest that jasmonate shifts the metabolic flux from flavonoids to stilbenes. DAP-Seq is an additional method to identify pathways if the common regulator is already known.


### (35) Additional flavonol biosynthesis regulators ###

> **_<a href="https://twitter.com/boas_pucker/status/1447449384862588931">TWEET</a>:_** MYB21 and its homologs MYB24/MYB57 activate the flavonol accumulation in Arabidopsis thaliana stamen #MybMonday (details: https://bit.ly/3BxImdv)

<a href="https://twitter.com/boas_pucker/status/1447449384862588931">
<img alt="Additional flavonol biosynthesis regulators (Tweet #35)" src="https://pbs.twimg.com/media/FBXNwIOWUAkaoA0?format=png&name=900x900" width="30%" height="30%">
</a>

[Zhang et al., 2021](https://doi.org/10.1093/jxb/erab156) investigated novel group of flavonol biosynthesis regulators around MYB21. MYB11/MYB12/MYB111 are well established regulators of the flavonol biosynthesis (MYB subgroup 7). MYB21/MYB24/MYB57 were reported as additional regulators that have a different phylogenetic origin (MYB subgroup 19). A _myb21_ mutant can be rescued by overexpression of FLS or feeding of the substrate. ROS accumulation is increased in the stamen of _myb21_ mutants suggesting that flavonols are required for mediating ROS stress.


### (36) MYB123 homologs of _Castanea crenata_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1449986100458446858">TWEET</a>:_** Castanea crenata has a large number of potential proanthocyanidin biosynthesis regulating MYBs suggesting a fine regulation of this pathway #MybMonday (details: https://bit.ly/3FUvvVy)

<a href="https://twitter.com/boas_pucker/status/1449986100458446858">
<img alt="MYB123 homologs of _Castanea crenata_ (Tweet #36)" src="https://pbs.twimg.com/media/FB6q-RSXEAUGHfO?format=jpg&name=large" width="30%" height="30%">
</a>

The annotation of the recently released genome sequence of Castanea crenata [Shirasawa,2021](https://doi.org/10.1093/dnares/dsab016) was screened for MYBs [Pucker, 2021](https://doi.org/10.1101/2021.10.16.464636). A large number of close homologs of AtMYB123 (TT2), a regulator of the proanthocyanidin biosynthesis, was discovered. This could indicate that the PA biosynthesis is controlled in a very specific way in different tissues and under different conditions. Tissue-specific expression analyes could help to resolve this question.


### (37) MYBs control coloration of blueberry and bilberry ###

> **_<a href="https://twitter.com/boas_pucker/status/1455074629907845120">TWEET</a>:_** MYBA1 activates MYBPA1.1 and both are required for anthocyanin regulation in blueberry and bilberry. MYBA1 shows high expression in the pigmented tissues, but weak or no expression in white tissues #MybMonday (details: https://bit.ly/3vVMh1D)

<a href="https://twitter.com/boas_pucker/status/1455074629907845120">
<img alt="MYBs control coloration of blueberry and bilberry (Tweet #37)" src="https://pbs.twimg.com/media/FCzRStOXEAMB1tc?format=jpg&name=large" width="30%" height="30%">
</a>

[Lafferty et al., 2021](https://doi.org/10.1093/jxb/erab460) investigated the regulation of anthocyanin and proanthocyanidin biosynthesis in blueberry (_Vaccinium corymbosum_) and bilberry (_V. myrtillus_). Blueberry has white flesh, while bilberry has red flesh indicating as difference in the anthocyanin biosynthesis regulation. While MYBPA1.1 and MYBA1 are both correlated with anthocyanin biosynthesis, they differ in their expression pattern. MYBPA1.1 is active during proanthocyanidin formation in early stages and anthocyanin formation during ripening. MYBA1 is a positive regulator of MYBPA1.1 which suggest that the lack of MYBA1 expression in blueberry flesh is the explanation for the lack of anthocyanin pigmentation. The authors speculate that such a hierarchical model could also be in place for other branches of the flavonoid biosynthesis.


### (38) MeJA induced MYB expression ###

> **_<a href="https://twitter.com/boas_pucker/status/1457739701960380424">TWEET</a>:_** Expression of Glycyrrhiza uralensis MYB4 and MYB88 is induced by methyl jasmonate leading to the activation of the flavonoid biosynthesis #MybMonday (details: https://bit.ly/3c7nKxT)

<a href="https://twitter.com/boas_pucker/status/1457739701960380424">
<img alt="MeJA induced MYB expression (Tweet #38)" src="https://pbs.twimg.com/media/FDrpC9vWQAUQGf0?format=jpg&name=medium" width="30%" height="30%">
</a>

[Li et al., 2020](https://doi.org/10.1371/journal.pone.0236565) investigated the influence of methyl jasmonate (MeJA) on suspended cells of _Glycyrrhiza uralensis_ via RNA-seq. The MeJA treatment resulted in increased C4H and CHS expression levels. Unfortunately, the orthologs of GlMYB4 and GlMYB88 in _Arabidopsis thaliana_ are not included in the phylogenetic analysis. Overexpression of MYB88 resulted in elevated gene expression of C4H and CHS. It seems that this also increased the total amount of produced flavonoids.



### (39) Purple sweet potato ###

> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** IbMYB1 regulates the purple pigmentation of leaf and storage root in sweet potato (Ipomoea batatas) #MybMonday (details: https://bit.ly/3qzsOTR)

<a href="https://twitter.com/boas_pucker/status/1460148059472478208">
<img alt="Purple sweet potato (Tweet #39)" src="https://pbs.twimg.com/media/FELsj_QXIA8pWRW?format=jpg&name=large" width="50%" height="50%">
</a>

[Zhang et al., 2021](https://doi.org/10.3389/fpls.2021.688707) investigated the influence of _IbMYB1_ on the pigmentation observed in different cultivars of sweet potato (_Ipomoea batatas_). Expression of this gene aligns well with the observed pigmentation patterns. Co-expression with _CHS_, _CHI_, _DFR_, _ANS_, and several _GTs_ was checked. While this MYB influences the pigmentation pattern, additional factors might contribute as well.


### (40) Automatic MYB annotation ###

> **_<a href="https://twitter.com/boas_pucker/status/1462684775840301068">TWEET</a>:_** Interested in MYB transcription factors? Try this pipeline for an automatic annotation of the MYBs in your species of interest: https://github.com/bpucker/MYB_annotator #MybMonday (details: https://doi.org/10.1101/2021.10.16.464636 & https://bit.ly/3catoPs)

<a href="https://twitter.com/boas_pucker/status/1462684775840301068">
<img alt="Automatic MYB annotation (Tweet #40)" src="https://pbs.twimg.com/media/FEL0WfbWUAAbCmJ?format=jpg&name=4096x4096" width="50%" height="50%">
</a>

Celebrating post number 40 in #MybMonday series by sharing the opportunity to get involved with MYBs. This pipeline allows the automatic identification and annotation of MYBs ([Pucker, 2021](https://doi.org/10.1101/2021.10.16.464636)). CDS or peptide sequence collections of a newly sequenced plant genomes or transcriptomes can be subjected to the analysis. This is a quick way (5-10 minutes) to get a solid annotation of the MYBs for in-depth investigations of the regulated processes. As a proof of concept, the MYB gene families of _Croton tiglium_ and _Castanea crenata_ were analyzed.


### (41) 4R-MYB AtSNAPc4 ###

> **_<a href="https://twitter.com/boas_pucker/status/1465221489335091202">TWEET</a>:_** 4R-MYB AtSNAPc4 is a member of a protein complex associated with snRNA gene transcription. Most plant species have just a single copy of this 4R-MYB #MybMonday (details: https://bit.ly/3D0xogK)

<a href="https://twitter.com/boas_pucker/status/1465221489335091202">
<img alt="4R-MYB AtSNAPc4 (Tweet #41)" src="https://pbs.twimg.com/media/FFUFRP9XEAASRUc?format=jpg&name=medium" width="30%" height="30%">
</a>

[Thiedig et al., 2021](https://doi.org/10.1093/plphys/kiaa067) investigated AtSNAPc4 (At3g18100), which is a 4R-MYB. In contrast to the abundant R2R3-MYBs, this protein is characterized by four and a half MYB repeats. This protein is a subunit of the SNAP complex 4 that is involved in snRNA gene transcription. _Arabidopsis thaliana_, _Beta vulgaris_, _Vitis vinifera_, and many other plants have only a single copy of this 4R-MYB. This suggest that this single copy state is conserved through evolution. Four different transcript isoforms of At3g18100 are annotated in Araport11, but only one of them (AtSNAPc4.1) encodes a functional protein. Therefore, it is possible that the other three isoforms are targeted by the NMD pathway.


### (42) Genome-wide MYB papers ###

> **_<a href="https://twitter.com/boas_pucker/status/1467758205371502593">TWEET</a>:_** Interested in MYBs? Here is a collection of studies that conducted genome-wide investigations of MYBs in plants: https://lnk.tu-bs.de/RQcHMD #MybMonday (details:https://bit.ly/3GjcsUd & https://bit.ly/3y1Ln4T)

<a href="https://twitter.com/boas_pucker/status/1467758205371502593">
<img alt="Genome-wide MYB papers (Tweet #42)" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/03/2020.02.03.932046/F2.large.jpg" width="30%" height="30%">
</a>

This [collection of papers](https://lnk.tu-bs.de/RQcHMD) about genome-wide investigations of the MYB gene family in plants shows the amount of work that has been done. The collection will be updated over time. Please get in touch if you know publications that are missing.


### (43) LoMYB33 ###

> **_<a href="https://twitter.com/boas_pucker/status/1470294919327715333">TWEET</a>:_** The R2R3-MYB LoMYB33 is a homolog of GAMYBs. Gene expression patterns and heterologous expression indicate that it is associated with the pollen development in lily #MybMonday (details: )

<a href="https://twitter.com/boas_pucker/status/1470294919327715333">
<img alt="LoMYB33 (Tweet #43)" src="https://pbs.twimg.com/media/FGcIH9pXwAQJdST?format=png&name=900x900" width="30%" height="30%">
</a>

[Liu et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2021.730007/full) investigated some MYBs in lily. A phylogenetic analysis suggest that LoMYB33 is a homolog of previously characterized GAMYBs of other plant species. LoMYB33 was found to be highly expressed in he late stages of anther development. Overexpression of LoMYB33 in Arabidopsis thaliana resulted in reduced growth and had a negative impact on anther development. This resulted in a lower number of pollen grains and partial male sterility.


### (44) R2R3-MYB C2 repressors in grapevine ###

> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** VvMYB4a and VvMYB4b repress the general phenylpropanoid pathway, while VvMYBC2-L1 and VvMYBC2-L3 repress the anthocyanin and proanthocyanidin biosynthesis #MybMonday (details: https://bit.ly/3lY8bxK)

<a href="https://github.com/bpucker/MybMonday">
<img alt="R2R3-MYB C2 repressors in grapevine (Tweet #44)" src="https://pbs.twimg.com/media/FATlMIEXMAEuEbE?format=jpg&name=medium" width="40%" height="40%">
</a>

[Cavallini et al., 2015](http://www.plantphysiol.org/cgi/doi/10.1104/pp.114.256172) investigated the phenylpropanoid repressors of _Vitis vinifera_. The identified repressor MYBs were similar to the previously chracterized flavonoid repressors AtMYB4 and AtMYBL2. The _V. vinifera_ sequences were classified into groups that are similar to AtMYB4 and PhMYB4 (VvMYB4a and VvMYB4b) and those that are similar to FaMYB1 and PhMYB27 (VvMYBC2-L1, VvMYBC2-L2, and VvMYBC2-L3). A further subclassification was achieved based on the presence of various C-motifs. Fig. 1 of the publication by Cavallini et al., 2015 is an excellent summary. Heterologous expression in petunia was used to study the functions of different repressors. VvMYB4a and VvMYB4b repressed the general phenylpropanoid biosynthesis in petunia. _VvMYBC2-L1_ and _VvMYBC2-L3_ expression resulted in reduced petal anthocyanin content and reduced amount of proanthocyanidins in seeds.


### (45) MYB function paper collection ###

> **_<a href="https://twitter.com/boas_pucker/status/1475368350494740480">TWEET</a>:_** Interested in the functions of individual MYBs? This collection of papers can help you to find functionally characterized orthologs: https://lnk.tu-bs.de/fX3in9 #MybMonday (details: )

<a href="https://twitter.com/boas_pucker/status/1475368350494740480">
<img alt="MYB function paper collection (Tweet #45)" src="https://pbs.twimg.com/media/E-BCE_RXEAElM5Y?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

This [collection of papers](https://github.com/bpucker/MybMonday/wiki/Collection-of-studies-characterizing-individual-MYBs) about the functions of individual MYBs is intended as a basis for future studies. Orthologs and closely related sequences with available functional data can be harnessed to inform phylogenetic analyses. Please get in touch if you know publications that are missing.


### (46) Automatic annotation of MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1477905064601681929">TWEET</a>:_** Starting 2022 with a new transcriptome/genome assembly? The automatic annotation of MYBs can help you to study a large transcription factor family: https://lnk.tu-bs.de/Jkg950 #MybMonday (details: https://bit.ly/31T8Au4)

<a href="https://twitter.com/boas_pucker/status/1477905064601681929">
<img alt="Automatic annotation of MYBs (Tweet #46)" src="https://pbs.twimg.com/media/E3Q59hoWEAA0RIj?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

The [MYB_annotator](https://github.com/bpucker/MYB_annotator) is freely available on github. This tool allows the identification and annotation of MYBs in a new species based on annotated coding or peptide sequences. The run time is between 5 and 10 minutes for most plant species.


### (47) VviMYB13 activates stilbene biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1480441780143222785">TWEET</a>:_** Grapevine VviMYB13 appears as a pivotal regulator of the stilbene biosynthesis in vegetative tissues and in response to downy mildew infection #MybMonday (details: https://lnk.tu-bs.de/H7oZiy)

<a href="https://twitter.com/boas_pucker/status/1480441780143222785">
<img alt="VviMYB13 activates stilbene biosynthesis (Tweet #47)" src="https://pbs.twimg.com/media/FIsadPcWQAY9p_1?format=jpg&name=medium" width="40%" height="40%">
</a>

[Wong et al., 2016](https://doi.org/10.1093/dnares/dsw028) performed a comprehensive analysis of the MYB transcription factors in grapevine. The annotation was manually improved based on information from RNA-seq datasets. Grapevines are well known for their production of stilbenes (anticarcinogenesis) and flavonoids (nutraceuticals). High production levels are based on gene duplications which include the regulating MYB transcription factors. MYBs associated with the flavonoid biosynthesis show patterns of selection that might be due to breeding over many generations. Several MYBs show alternative transcripts originating from the retention of introns. Many of these transcripts encode a disrupted DNA binding domain. Co-expression analyses were preformed to understand the functions of the grapevine MYBs. An enrichment among the most highly co-expressed genes of each MYB was used to predict a function. _VviMYB13_ was previously not characterized, but appeared co-expressed with the stilbene synthase activators _VviMYB14_ and _VviMYB15_. The expression domain of _VviMYB13_ is not identical with that of its close homolog _VviMYB15_. _VviMYB13_ expression in vegetative tissues under non-stress conditions appears to be responsible for the activation of stilbene biosynthesis. _VviMYB13_ might also be important for the activation of stilbene biosynthesis at the onset of ripening before _VviMYB14_ and _VviMYB15_ are active. _VviMYB13_ also showed a substantially faster and higher response to downy mildew infection than _VviMYB14_ or _VviMYB15_.
Check out the website of [TOMSBio lab](https://tomsbiolab.com/) to learn more about (grapevine) MYBs and their target pathways.




### (48) Apple skin color mutant ###

> **_<a href="https://twitter.com/boas_pucker/status/1482978494808162305">TWEET</a>:_** MdMYB90-like activates anthocyanin biosynthesis in apple. Methylation changes in the promoter region of a mutant are presented as explanation for an enhanced MdMYB90-like expression #MybMonday (details: https://lnk.tu-bs.de/RXK1aW)

<a href="https://twitter.com/boas_pucker/status/1482978494808162305">
<img alt="Apple skin color mutant (Tweet #48)" src="https://pbs.twimg.com/media/FJRHiLCWYAUnDOT?format=jpg&name=medium" width="40%" height="40%">
</a>

[Sun et al., 2021](https://doi.org/10.1038/s41438-021-00590-3) investigated an apple mutant that shows an enhanced skin coloration. RNA-seq revealed that a MYB90-like transcripiton factor is a putative key regulator due to substantially increased gene expression. This TF was able to bind to the promoters of _MdCHS_ and _MdUFGT_. Overexpression of this TF resulted in activation of anthocyanin-related genes. As no causal sequence variant was identified in this MYB, the authors assume that a difference in the methylation of the promoter region of this MYB is responsible for the expression difference.



### (49) Soybean seed coat pigmentation ###

> **_<a href="https://twitter.com/boas_pucker/status/1485522760356941824">TWEET</a>:_** GmMYBA2 activates and GmMYBR represses the anthocyanin biosynthesis in soybean. Both appear to form a feedback loop to fine-tune seed coat pigmentation #MybMonday (details: https://lnk.tu-bs.de/CnuV1T)

<a href="https://twitter.com/boas_pucker/status/1485522760356941824">
<img alt="Soybean seed coat pigmentation (Tweet #49)" src="https://pbs.twimg.com/media/FJ2h01gXwAEts0r?format=jpg&name=4096x4096" width="40%" height="40%">
</a>


[Gao et al., 2021](https://doi.org/10.1093/jxb/erab152) investigated the transcriptional regulation of the seed coat pigmentation in soybean (_Glycine max)_. Soybean is an important crop species for the production of oil and protein. Seeds show a complex pigmentation pattern. Especially dark seeds are expected to have health benefits. A phylogenetic analysis connected 15 MYB genes with the anthocyanin biosynthesis. GmMYBR belongs in a clade with C2 repressor MYBs. GmMYBA2 is a member of the anthoyanin activator clade. Expression patterns and transactivation assays suggested that GmMYBR is a repressor and GmMYBA2 is an activator of the anthocyanin biosynthesis. Overexpression of GmMYBA2 in the soybean cultivars Wm82 and P3 resulted in the accumulation of cyanidin derivatives and delphinidin derivatives, respectively. This suggest a difference in the F3'5'H activity between bloth cultivars. A single nucleotide deletion in the _F3'H_ of P3 caused a frameshift and resulted in a probably non-functional truncated protein. The _F3'5'H_ of Wm82 is probably also non-functional due to a premature stop codon. GmMYBR seems to repress the gene expression by binding the GmMYBA2-GmTT8a complex that would otherwise activate anthocyanin biosynthesis genes.



### (50) Celery MYB12 activates apigenin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1488044375049338885">TWEET</a>:_** Celery MYB12 is a positive regulator of the apignenin and luteolin biosynthesis through activation of PAL, C4H, 4CL, CHS, CHI, and FNS. #MybMonday (details: https://lnk.tu-bs.de/S30czt)

<a href="https://twitter.com/boas_pucker/status/1488044375049338885">
<img alt="Celery MYB12 activates apigenin biosynthesis (Tweet #50)" src="https://pbs.twimg.com/media/FKIs_WZWQAsCMIR?format=jpg&name=large" width="40%" height="40%">
</a>


[Wang et al., 2021](https://doi.org/10.1007/s00299-021-02792-4) identified the bHLH-independent R2R3 protein MYB12 as a regulator of FNS in celery (_Apium graveolens_). AgMYB12 contains the SG7 and SG7-2 motifs that are characteristic for this MYB clade. The binding of AgMYB12 to the promoter of AgFNS was shown via Y2H. AgMYB12 was found to be positively correlated with the apigenin and luteolin content. Heterologous expression of AgMYB12 in celery plants did not change the visible phenotype. However, apigenin and luteolin levels were increased in these overexpression lines. Target genes are _PAL_, _C4H_, _4CL_, _CHS_, _CHI_, and _FNS_ as discovered based on increased expression in the transgenic celery lines. Gene expression of _F3H_ was not meassured. It remains unclear if the activated FNS belongs to the FNSI or FNSII lineage. FNSI appears to be unique to the Apiaceae family.



### (51) VcMYB1 activates anthocyanin biosynthesis in blueberry ###

> **_<a href="https://twitter.com/boas_pucker/status/1490581090276298754">TWEET</a>:_** The subgroup6 member VcMYB1 activates the anthocyanin biosynthesis in blueberry #MybMonday (details: https://lnk.tu-bs.de/qRVcPY)

<a href="https://twitter.com/boas_pucker/status/1490581090276298754">
<img alt="VcMYB1 activates anthocyanin biosynthesis in blueberry (Tweet #51)" src="https://pbs.twimg.com/media/FK8qjpDXEAAGUAz?format=png&name=900x900" width="40%" height="40%">
</a>

[Tang et al., 2021](https://pubmed.ncbi.nlm.nih.gov/34659323/) identified VcMYB1 based on an expression pattern that matches the emergence of pigments over time. Additionally, the sequence of this MYB is very similar to previously characterized anthocyanin regulators. Heterologous expression of _VcMYB1_ in _Arabidopsis thaliana_ increased the anthocyanin content by 110x. Binding to the _DFR_ promoter was demonstrated. Transcriptional control of the anthocyanin biosynthesis in blueberry was also recently studied by [Lafferty et al., 2021](https://doi.org/10.1093/jxb/erab460).



### (52) MdMYB10 as betalain activator? ###

> **_<a href="https://twitter.com/boas_pucker/status/1493117807986241536">TWEET</a>:_** Is the anthocyanin activator MdMYB10 able to activate the betalain biosynthesis when expressed in sugar beet? #MybMonday (details: https://lnk.tu-bs.de/JOrE9B)

<a href="https://twitter.com/boas_pucker/status/1493117807986241536">
<img alt="MdMYB10 as betalain activator? (Tweet #52)" src="https://pbs.twimg.com/media/FLghP66XEAQsEcr?format=jpg&name=4096x4096" width="40%" height="40%">
</a>


[Khademi et al., 2022](https://doi.org/10.1007/s13205-022-03120-7) transformed _MdMYB10_ into sugar beet to activate the betalain biosynthesis. MdMYB10 is a known regulator of the anthocyanin biosynthesis in apple. This MYB can bind its own promoter resulting in an autoactivation ([Espley et al., 2009](https://doi.org/10.1105/tpc.108.059329)). Anthocyanins and betalains are two of the most important pigment groups in plants. While anthocyanins are almost ubiquitous, betalains occur only in the Caryophyllales (reviewed by [Timoneda et al., 2019]( https://doi.org/10.1111/nph.15980)). Anthocyanins and betalains are considered mutually exlcusive, because no solid study was able to detect both pigments in the same plant during the last 50 years. MYBs (e.g. MYB75=PAP1) are usually used to activate the anthocyanin biosynthesis in biotechnological applications. The activation of the anthocyanin biosynthesis and the resulting pigmentation can be used as a screening marker when transforming plants. However, this activation of the anthocyanin biosynthesis requires a bHLH and TTG1 as partners for the MYB. The PAP1 homolog in sugar beet was identified as a bHLH-independent transcriptional activator of the betalain biosynthesis ([Hatlestadt et al., 2015](https://rdcu.be/cGV9u)). Kahademi et al. show that sugar beet plantlets harboring the _MdMYB10_ transgene accumulate red pigments after 50 days when exposed to light. The authors conclude that the MdMYB10 activates the betalain biosynthesis in sugar beet, but also state that there is large variation between different transgenic lines. Unfortunately, the gene expression of the betalain biosynthesis genes was not analyzed. Such an analysis would have been important. It seems that _MdMYB10_ was successfully expressed in sugar beet, but does it activate the betalain biosynthesis or is the betalain formation caused by a native regulator? The production of betalains was shown to have an inhibitory effect on bacterial grotwh. This inhibition was demonstrated with leaf extracts on an agar plate.


### (53) Radiation protection 'plan B' ###

> **_<a href="https://twitter.com/boas_pucker/status/1495662069965803531">TWEET</a>:_** VvMYB24 activates terpene & flavonol biosynthesis in berry skin parts that lack anthocyanin pigementation #MybMonday (details: https://lnk.tu-bs.de/pnXkRE)

<a href="https://twitter.com/boas_pucker/status/1495662069965803531">
<img alt="Radiation protection 'plan B' (Tweet #53)" src="https://pbs.twimg.com/media/FIsadPcWQAY9p_1?format=jpg&name=medium" width="40%" height="40%">
</a>

[Chen et al., 2021](https://doi.org/10.1101/2021.12.16.472692 ) compared white/pale and anthocyanin-pigmented skins sections of _Vitis vinifera_ berries. Sections without anthocyanins are characterized by non-functional alleles of the anthocyanin regulators _VvMYBA1_ and _VvMYBA2_. Since anthocyanins are crucial for the pigmentation, these alleles explain the lack of color. _VvMYBA1_ is repressed by a transposon insertion in the promoter that can be lost through somatic mutations hence leading to local pigmentation of white berries. VvMYB24 (VIT_14s0066g01090), a subgroup 19 member, was studied through DAP-seq. This investigation resulted in the identification of _HY5_ and _MYC2_ as important targets. These results suggest that VvMYB24 activates the flavonol biosynthesis and the terpene (carotenoid) biosynthesis. This study is additional evidence for the sunscreen function of anthocyanins.


### (54) PgMYB308-like of pomegranate ###

> **_<a href="https://twitter.com/boas_pucker/status/1498198786325106688">TWEET</a>:_** PgMYB308-like of pomegranate activates shikimate pathway and lignin biosynthesis, but represses the biosynthesis of flavonoids #MybMonday (details: https://lnk.tu-bs.de/DcKsAw)

<a href="https://twitter.com/boas_pucker/status/1498198786325106688">
<img alt="PgMYB308-like of pomegranate (Tweet #54)" src="https://pbs.twimg.com/media/FMjfBLnXMAAbRIr?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

[Dhakarey et al., 2022](https://doi.org/10.1093/hr/uhac008) investigated the function of PgMYB308-like in _Punica granatum_ (pomegranate). Pomegranate juice is associated with many health benefits which were attributed to phenolic compounds with antioxidant activity. Various flavonoids and other specialized metabolites are expected to play important roles. The biosynthesis of these compounds depends on the shikimate pathway for formation of the substrate phenylalanine. R2R3-MYB transcription factors are known to play important roles in the regulation of these biosynthesis pathways. PgMYB308-like was identified as a subgroup4 R2R3-MYB through a phylogenetic anlysis. Members of this subgroup have been reported as flavonoid repressors in previous studies. A motifs for the interaction with bHLH partners was discovered in PgMYB308-like. Overexpression of this _PgMYB308-like_ in pomegranate hairy root cells resulted in an increased production of aromatic amino acids and lignins, but reduced formation of flavonoids. The production of tyrosine and tryptophan was substantially stronger increased than the formation of phenylalanine. Among the up-regulated target genes of PgMYB308-like are caffeic acid 3-O-methyltransferase (COMT), laccase 7-like, and laccase 13. It appears that PgMYB308-like is not just activating the lignin biosynthesis, but also regulates the cell wall formation. Down-regulated genes include MYB123-like, MYB308, and EGL1-like that are involved in the flavonoid biosynthesis.


### (55) Poplar MYB117 promotes anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1500735500033970177">TWEET</a>:_** Poplar MYB117 activates the anthocyanin biosynthesis and increases the hydroxylation of flavonoids through enhanced expression of F3'5'H #MybMonday (details: https://lnk.tu-bs.de/9pMGmz)

<a href="https://twitter.com/boas_pucker/status/1500735500033970177">
<img alt="Poplar MYB117 promotes anthocyanin biosynthesis (Tweet #55)" src="https://pbs.twimg.com/media/FNMBN5EXoAco-2q?format=jpg&name=medium" width="40%" height="40%">
</a>

[Ma et al., 2021](https://doi.org/10.1093/jxb/erab116) investigated the role of poplar MYB117 in _Populus tremula x tremuloides_. There are six PAP1-type MYBs in poplar: MYB113, MYB116, MYB117, MYB118, MYB119, and MYB120. MYB117 does not act on its own, but requires the formation of a complex with bHLH131 for this activation. This bHLH partner is also forming complexes with the proanthocyanidin regulators MYB115 and MYB134. It seems that this bHLH does not have a preference for any of these three MYBs. Overexpression of MYB117 in poplar caused an increase of the anthocyanin content by >50-fold while the proanthocyanidin content was only slightly increased. Expression of the core flavonoid biosynthesis genes was activated by MYB117, but also some TFs showed up-regulation. The MYB117 paralog MYB113 showed a striking up-regulation which indicated that this might be a direct target of MYB117. Both _F3'5'H_ genes and _b5_ showed a substantial up-regulation. High F3'5'H activity resulted in a higher ratio of delphinidin derivatives to cyanidin derivatives (about 1:1) compared to the wild type (about 1:10). The increase in the prodelphinidin to procyanidin ratio was similar, but at a lower level. The same was observed for flavonols.



### (56) Repressor function of MYB12 ###

> **_<a href="https://twitter.com/boas_pucker/status/1503272215575412736">TWEET</a>:_** MYB12 binds TMO5 resulting in reduced activity of the TMO5/LHW complex. Expression of the target genes is reduced. MYB12 is an indirect TMO5/LHW target. This system constitutes a negative feedback loop #MybMonday (details: https://lnk.tu-bs.de/r72Qy6)

<a href="https://twitter.com/boas_pucker/status/1503272215575412736">
<img alt="Repressor function of MYB12 (Tweet #56)" src="https://pbs.twimg.com/media/FNykhr-WUAESi6Y?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

[Wybouw et al., 2022](https://doi.org/10.1101/2022.03.08.483486) investigated the role of MYB12 as repressor of the TMO5/LHW-mediated gene expression. The bHLH transcription factors TARGET OF MONOPTEROS 5 (TMO5) and LONESOME HIGHWAY (LHW) form a heterodimer that controls vascular cell proliferation that is responsible for radial expansion of the vascular bundle. High TMO5/LHW activity results in shorter roots with increased width. TMO5/LHW activates the expression of LONELY GUY 3(LOG3) and BETA GLUCOSIDASE 44 (BGLU44) which are involved in the cytokinin formation. SUPPRESSOR OF ACAULIS51-LIKE (SACL) is a repressor of this process due to competition with TMO5 for binding LHW. EMS mutagenesis generated a novel _myb12_ allele which revealed the novel repressor function of MYB12. The authors identified MYB12 as an additional repressors that binds TMO5 and prevents the formation/activity of the bHLH heterodimer. As MYB12 is also an indirect target of TMO5/LHW, this system consitutes a negative feedback loop. The molecular mechanism of the repression by MYB12 remains elusive. One hypothesis suggest that MYB12 might redirects the activity of the TMO5/LHW complex to other target genes.



### (57) Automatic identification and annotation of MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1505786564223410179">TWEET</a>:_** MYBs can be identified and annotated automatically (https://rdcu.be/cJm7Q). The corresponding #Python code and required data sets are freely available @github. A web server is under development. #MybMonday: https://lnk.tu-bs.de/ZwFMei

<a href="https://twitter.com/boas_pucker/status/1505786564223410179">
<img alt="Automatic identification and annotation of MYBs (Tweet #57)" src="https://pbs.twimg.com/media/FOWgj2AX0AIsZUD?format=jpg&name=large" width="40%" height="40%">
</a>

[Pucker, 2022](https://doi.org/10.1186/s12864-022-08452-5) describes the automatic identification and functional annotation of MYBs. The Python tool is available for download: [MYB_annotator](https://github.com/bpucker/MYB_annotator). A web server for more convenient application is under development. Get in touch if you need help running a MYB analysis.


### (58) Updated collection of MYB papers ###

> **_<a href="https://twitter.com/boas_pucker/status/1508330546589487104">TWEET</a>:_** Interested in the biological functions of individual MYBs? Here is an updated collection of MYB papers: https://lnk.tu-bs.de/fX3in9 #MybMonday (details: https://lnk.tu-bs.de/7cpPBe)

<a href="https://twitter.com/boas_pucker/status/1508330546589487104">
<img alt="Updated collection of MYB papers (Tweet #58)" src="https://pbs.twimg.com/media/E-BCE_RXEAElM5Y?format=jpg&name=small" width="40%" height="40%">
</a>



### (59) Bamboo MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1510867262181322752">TWEET</a>:_** Investigation of the MYBs in the fast growing woody Phyllostachys edulis (bamboo) revealed some potential regulators of the secondary cell wall development #MybMonday (details: https://lnk.tu-bs.de/ty37LB)


<a href="https://twitter.com/boas_pucker/status/1510867262181322752">
<img alt="Bamboo MYBs (Tweet #59)" src="https://pbs.twimg.com/media/FPeDEVoX0AkNQbU?format=jpg&name=large" width="40%" height="40%">
</a>

[Yang et al., 2019](https://doi.org/10.7717/peerj.6242) investigated the MYB gene family in moso bamboo (_Phyllostchys edulis_). A total of 85 MYBs was identified and characterized by phylogenetic analysis, gene structure analysis, and gene expression analysis. This set of MYBs includes 82 R2R3-MYBs. The expression pattern of 12 MYBs suggests that they might be associated with the biosynthesis of the secondary cell wall (SCW).
Lignified SCWs are of biotechnological interest, because they are responsible for the wood quality. Bamboo is characterized by rapid growth which makes it an economically important species. The identification of MYBs associated with the wood quality provides the basis for modifications through genetic engineering or breeding in the future. While this study revealed some candidates, additional analyses are required to understand how the development of SCW is regulated in bamboo.



### (60) Evolution of flavonol regulators in _Brassicaceae_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1513403976334528515">TWEET</a>:_** Exciting findings: evolution of flavonol biosynthesis regulating MYBs in Brassicaceae involves a deep MYB12 and MYB111 duplication as well as a MYB11 loss #MybMonday: https://doi.org/10.1101/2022.04.06.487363 & https://lnk.tu-bs.de/B3wTUb

<a href="https://twitter.com/boas_pucker/status/1513403976334528515">
<img alt="Evolution of flavonol regulators in _Brassicaceae_ (Tweet #60)" src="https://pbs.twimg.com/media/FQAiZohWUAEsg-5?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

[Schilbert & Glover, 2022](https://doi.org/10.1101/2022.04.06.487363) performed comprehensive analyses of the MYB gene family in the _Brassicaceae_. The flavonol biosynthesis in _Arabidopsis thaliana_ is regulated by MYB11/MYB12/MYB111 which have partially redundant functions. A deep duplication of _MYB12_ and _MYB111_ was discovered. In addition, loss of _MYB11_ was identified. These findings are supported by synteny and phylogenetic analyses.



### (61) Anthocyanin regulation in _Paeonia ostii_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1515940692685729794">TWEET</a>:_** Flower colors of Paeonia ostii are determined by anthocyanin concentrations and range from white to pink. PoMYB2 and PoSPL1 were proposed as negative regulators of the anthocyanin biosynthesis #MybMonday (details: https://lnk.tu-bs.de/O8gWwC)

<a href="https://twitter.com/boas_pucker/status/1515940692685729794">
<img alt="Anthocyanin regulation in _Paeonia ostii_ (Tweet #61)" src="https://pbs.twimg.com/media/FQRlYF9WUAQ-Lg9?format=jpg&name=large" width="40%" height="40%">
</a>

[Gao et al., 2016](https://doi.org/10.3389/fpls.2015.01205) investigated the molecular basis of flower color variation in _Paeonia ostii_. Flower colors of this species range from white to pink. The color is determined by the amount of accumulated anthocyanins. The transcription factors PoMYB2 and PoSPL1 were identified as potential negative regulators of the anthocyanin biosynthesis. It is spectulated that these transcription factors reduce the activation capacity of the MYB-bHLH-TTG1 complex that activates the anthocyanin biosynthesis.


### (62) MYB_annotator online ###

> **_<a href="https://twitter.com/boas_pucker/status/1518477407346302983">TWEET</a>:_** Automatic MYB annotation is now available as an online service:
http://pbb.bot.nat.tu-bs.de/MYB_annotator/. Just submit a FASTA file of your species of interest to start the analysis. #MybMonday (details:  https://bit.ly/3xMTMLl & https://doi.org/10.1186/s12864-022-08452-5)

<a href="https://twitter.com/boas_pucker/status/1518477407346302983">
<img alt="MYB_annotator online (Tweet #62)" src="https://pbs.twimg.com/media/FRJ9-rGX0AAjU2H?format=jpg&name=large" width="40%" height="40%">
</a>

The automatic MYB annotation workflow ([Pucker, 2022](https://doi.org/10.1186/s12864-022-08452-5)) is  now available as an online service: [MYB_annotator online](http://pbb.bot.nat.tu-bs.de/MYB_annotator/). Users can submit a FASTA file with peptide sequences of their species of interest without having to install the tool locally.


### (63) Regulation of anthocyanin biosynthesis in petunia ###

> **_<a href="https://twitter.com/boas_pucker/status/1521014122288189440">TWEET</a>:_** Petunia exserta lacks a functional AN2 (anthocyanin regulating MYB). A paralog MYB resurrects anthocyanin pigmentation in a white flowering lineage and shifts flower color to red #MybMonday (details: https://lnk.tu-bs.de/ysaXNg)

<a href="https://twitter.com/boas_pucker/status/1521014122288189440">
<img alt="Regulation of anthocyanin biosynthesis in petunia (Tweet #63)" src="https://pbs.twimg.com/media/FRqYl-jX0AEA2iI?format=jpg&name=large" width="40%" height="40%">
</a>

[Berardi et al., 2021](https://doi.org/10.1093/plcell/koab114) investigated the molecular basis of red floral color in _Petunia exserta_. Red flower color evolved multiple times independently from purple coloration through loss-of-function mutations. One major driver of this evolution is the pollination by hummingbirds. However, _Petunia exserta_ evolved from a white flowering ancestor. The key anthocyanin regulator _AN2_ is non-functional in this species. A paralog of _AN2_ restored the anthocyanin pigmentation. Hydroxylation patterns are modified and acylation of anthocyanins is prevented. Different _F3'5'H_ paralogs are highly expressed in _P. exserta_ (red) and _P. secreta_ (purple) and are likely to act upon different substrates. Low expression of _AAT_ prevents down-stream modifications of anthocyanins. There are four anthocyanin regulating MYBs in long-tube petunia (_AN2_, _AN4_, _DPL_, _PHZ_) which control the pigmentation of defined floral structures. The search for the key anthocyanin regulator in _P. exserta_ revealed _DPL_ and _PH4_ as most suitable candidates based on their expression patterns. Repression of _DPL_caused white flowers exclusively in _P. exserta_, while repression of _PH4_ effects multiple _Petunia_ species. The authors assume that DPL shifted from activation of anthocyanin biosynthesis in the veins of the floral tube to activation of the anthocyanin biosynthesis in the limb. This is a rare example of neofunctionalization instead of subfunctionalization. PH4 might be involved in activating the anthocyanin transport genes. A loss-of-function mutation in the flavonol regulating _MYB-FL_ removes the competition of flavonol and anthocyanin biosynthesis for substrate thus enabeling anthocyanin pigmentation.



### (64) Regulation of anthocyanin biosynthesis in kiwifruit ###

> **_<a href="https://twitter.com/boas_pucker/status/1523550836202229760">TWEET</a>:_** MYB10 and MYB110 activate anthocyanin biosynthesis in Actinidia chinensis. While MYB110 causes pigment accumulation in the entire fruit, MYB10 leads to pigmentation of the inner pericarp #MybMonday (details: )

<a href="https://twitter.com/boas_pucker/status/1523550836202229760">
<img alt="Regulation of anthocyanin biosynthesis in kiwifruit (Tweet #64)" src="https://pbs.twimg.com/media/FSDN9zRaMAE0JYm?format=jpg&name=large" width="40%" height="40%">
</a>

[Wang et al., 2022](https://doi.org/10.1111/nph.18122) studied the differential expression of the anthocyanin regulators MYB10 and MYB110 in _Actinidia chinensis_ (kiwifruit). This species is characterized by a variety of differently pigmented cultivars including green, yellow, red, and purple. The gene expression difference of these MYBs is controlled at the transcriptional level by miRNAs that degrade repressors of these MYBs. MYB10 and MYB110 are both able to activate the anthocyanin biosynthesis. MYB210 is phylogenetically related to MYB10 and MYB110, but appears to be non-functional. MYB10 and MYB210 appear to be the result of a whole genome duplication. MYB210 and MYB110 seem to be tandem duplicates. MYB110 appears to be a stronger activator than MYB10. While _MYB10_ overexpression only resulted in anthocyanin formation in the inner pericarp, overexpression of _MYB110_ causes anthcocyanin formation across the fruit.


### (65) Regulation of proanthocyanidin biosynthesis in banana ###

> **_<a href="https://twitter.com/boas_pucker/status/1526087551152431105">TWEET</a>:_** Proanthocyanidin biosynthesis in banana is activated by MaMYBPA1 and MaMYBPA2. Four other MYBs serve as repressors #MybMonday (details: https://lnk.tu-bs.de/MxnLYC)


<a href="https://twitter.com/boas_pucker/status/1526087551152431105">
<img alt="Regulation of proanthocyanidin biosynthesis in banana (Tweet #65)" src="https://pbs.twimg.com/media/FS1t1heX0AIjqTi?format=jpg&name=large" width="40%" height="40%">
</a>

[Rajput et al., 2022](https://doi.org/10.1101/2022.04.27.489641) investigated the proanthocyanidin biosynthesis regulating MYB transcription factors in _Musa acuminata_. MaMYBPA1 and MaMYBPA2 activate the proanthocyanidin biosynthesis, while MaMYBR1-MaMYBR4 serve as repressors. Applied methods include the complementation of _Arabidopsis thaliana_ _tt2_ mutants and promoter binding assays. MaMYBs interacted with bHLH and TTG1 to form a functional MBW complex.


### (66) MIXTA-MYB in _Marchantia polymorpha_ ###

> **_<a href="https://twitter.com/boas_pucker/status/1528609171393982464">TWEET</a>:_** MIXTA-MYB MpSBG9 regulates the cutin biosynthesis in Marchantia polymorpha. This suggest a role of the MIXTA-MYB lineage in the evolution of the cuticle #MybMonday (https://lnk.tu-bs.de/7DW1me & https://doi.org/10.1111/nph.16997)

<a href="https://twitter.com/boas_pucker/status/1528609171393982464">
<img alt="MIXTA-MYB in _Marchantia polymorpha_ (Tweet #66)" src="https://pbs.twimg.com/media/FTW6hiKXwAM_VHX?format=jpg&name=large" width="40%" height="40%">
</a>

[Xu et al., 2020](https://doi.org/10.1111/nph.16997) characterized the subgroup 9 R2R3-MYB transcription factor MpSBG9 in _Marchantia polymorpha_. This MYB is homologous to MIXTA-MYBs of vascular plants and regulates the biosynthesis of cutin, but not the wax biosynthesis. The discovery of the MpSBG9 functions suggests an involvement of the MIXTA-MYB lineage in the evolution of the cuticle.


### (67) MYB93 is involved in lateral root development ###
> **_<a href="https://twitter.com/boas_pucker/status/1531160982244020224">TWEET</a>:_** MYB93 regulates lateral root development in Arabidopsis in response to very long fatty acid levels. myb93 mutants showed faster lateral root development, while MYB93 overexpression slowed down the process #MybMonday (details: https://lnk.tu-bs.de/0UGNXW)

<a href="https://twitter.com/boas_pucker/status/1531160982244020224">
<img alt="MYB93 is involved in lateral root development (Tweet #67)" src="https://pbs.twimg.com/media/FT8kb_MX0AEJLWs?format=jpg&name=large" width="40%" height="40%">
</a>

[Uemura et al., 2022](https://doi.org/10.1101/2022.05.24.493307) investigated the role of _Arabidopsis thaliana_ MYB93 in the regulation of lateral roots in response to very long chain fatty acid (VLCFA) levels. MYB93 was found to regulate the expression of cell wall remodling genes including expansins. Expression of expansins is repressed by MYB93. VLCFA treatment caused a reducation of the _MYB93_ expression levels. _MYB93_ was specifically expressed in the endodermis of LROCs with an increase from stage I to stage V. However, _MYB93_ expression was not observed in lateral root primordia (LRPs). Knock-out of _MYB93_ lead to a faster development of LRPs, while overexpression of _MYB93_ resulted in a slower development of lateral roots. 


### (68) MtLAP1 controls anthocyanin biosynthesis in Medicago ###
> **_<a href="https://twitter.com/boas_pucker/MybMonday">TWEET</a>:_** The anthocyanin regulating MYB MtLAP1 controls the expression of MtGSTF7 in Medicago. Expression of MtGSTF7 is crucial for anthocyanin accumulation in the vacuole #MybMonday (details: https://lnk.tu-bs.de/03Q7RX)

<a href="https://twitter.com/boas_pucker/status/1533697698318065666">
<img alt="MtLAP1 controls anthocyanin biosynthesis in Medicago (Tweet #68)" src="https://pbs.twimg.com/media/FUTj12DXsAAcVzq?format=jpg&name=large" width="40%" height="40%">
</a>

[Wang et al., 2022](https://doi.org/10.1093/jxb/erac112) investigated the up-regulated genes in LAP1 overexpression _Medicago truncatula_ plants. While some of these genes are known members of the anthocyanin biosynthesis, novel genes were identified. _MtGSTF7_ was among these genes. Expression of _MtGSTF7_ was also identified to correlate with the deposition of anthocyanins. The _gstf7_ mutant lacks anthocyanins thus indicating that this genes is crucial for the pigmentation with anthocyanins. Activation of _MtGSTF7_ by LAP1 was predicted by PlantCARE and validated by dual luciferase assays. These results align with previous research on soybean which discovered differences in the target genes of anthocyanin and proanthocyanidin biosynthesis regulators.



### (69) _Malus baccata MYB4_ ###
> **_<a href="https://twitter.com/boas_pucker/MybMonday">TWEET</a>:_** Heterologous expression of _Malus baccata MYB4_ in _A. thaliana_ seedlings increases the cold and drought stress tolerance #MybMonday (details: https://lnk.tu-bs.de/2xLYjQ)

<a href="https://twitter.com/boas_pucker/status/1536234412236341248">
<img alt="_Malus baccata MYB4_ (Tweet #69)" src="https://pbs.twimg.com/media/FVEsV00WYAEmmI6?format=jpg&name=large" width="40%" height="40%">
</a>

[Yao et al., 2022](https://doi.org/10.3390/ijms23031794) investigated the function of _Malus baccata MYB4_ through heterologous expression in _Arabidopsis thaliana_ seedlings. _MbMYB4_ is the ortholog of _MYB4_ in _Malus domestica_. _MYB4_ shows increased expression in apple under cold stress, salt stress, and drought stress. The authors speculate that the activation of a CBF-dependent pathway might explain the observed phenotype. The drought response genes _CAT1_ and _P5CS_ were also reported as target genes of _MYB4_.


### (70) _Brassica napus MYB28_ determines seed GSL content ###
> **_<a href="https://twitter.com/boas_pucker/status/1538771126637166594">TWEET</a>:_** Brassica napus MYB28 determines the glucosinolate (GSL) content of seeds through regulation of the biosynthesis. Disruption of the ORF leads to low GSL lines #MybMonday XXXXX & https://doi.org/10.1101/2022.06.01.494149 

<a href="https://twitter.com/boas_pucker/status/1538771126637166594">
<img alt="_Brassica napus MYB28_ determines seed GSL content (Tweet #70)" src="https://pbs.twimg.com/media/FVqylwEX0AEQiWF?format=jpg&name=4096x4096" width="40%" height="40%">
</a>

[Schilbert et al., 2022](https://doi.org/10.1101/2022.06.01.494149) identified MYB28 as a crucical factor determining the glucosinolate (GSL) content in a segregating _Brassica napus_ (rapeseed) population. Mapping-by-sequencing revealed a genomic locus harbouring a MYB28 homeolog. This gene is highly expressed in seeds. Low GSL lines are characterized by a 4bp insertion in the open reading frame of this gene wich results in a disruptive frameshift mutation. The resulting truncated MYB28 is no longer able to activate the GSL biosynthesis.


### (71) _Ipomoea nil_ R2R3-MYBs ###
> **_<a href="https://twitter.com/boas_pucker/status/1541307844623867905">TWEET</a>:_** Several R2R3-MYBs of the C16 lineage are proposed as regulators of the anthocyanin biosynthesis in Japanese morning glory (Ipomoea nil). An intraspecific variation affects one of these putative anthocyanin regulators #MybMonday (details: https://lnk.tu-bs.de/1hn3cd)

<a href="https://twitter.com/boas_pucker/status/1541307844623867905">
<img alt="_Ipomoea nil_ R2R3-MYBs (Tweet #71)" src="https://pbs.twimg.com/media/FWMLLbNXgAA7VSY?format=jpg&name=large" width="40%" height="40%">
</a>

[Komatsuzaki et al., 2022](https://doi.org/10.1101/2022.06.23.497360) performed a genome-wide investigation of R2R3-MYBs in Japanese morning glory (_Ipomoea nil_). InMYB1 (INIL00g10723, INIL00g10723, INIL11g4087) was previously identified as an anthocyanin regulator, because its knock-out caused a flower mutant phenotype. InMYB2 (INIL05g09650) and InMYB3 (INIL05g09651) have also been considered as potential anthocyanin regulators. Clade 15 of the InMYBs corresponds to _Arabidopsis thaliana_ subgroup 7 thus most likely being involvedin flavonol regulation. Clade 16 of the IbMYBs matches the _A. thaliana_ subgroup 6 which is responsible for the anthocyanin regulation. Therefore, five genes in C16 are potential anthocyanin regulators. InMYB3 (INIL05g09651) seems to have an interspecific polymorphism as it lacks a complete R2R3 domain in this study, but was reported as a complete MYB before. 


### (72) _Brassica rapa_ _MYB18.1_ ###
> **_<a href="https://twitter.com/boas_pucker/status/1543844557179297794">TWEET</a>:_** A LTR/Copia insertion in Brassica rapa MYB18 shows a strong allele frequency difference between heading and nonheading populations indicating strong selection during domestication #MybMonday (details: https://lnk.tu-bs.de/9kHfzb)

<a href="https://twitter.com/boas_pucker/status/1543844557179297794">
<img alt="_Brassica rapa_ _MYB18.1_ (Tweet #72)" src="https://pbs.twimg.com/media/FWytuavX0AMFU4z?format=jpg&name=medium" width="40%" height="40%">
</a>

[Cai et al., 2022](https://onlinelibrary.wiley.com/doi/epdf/10.1111/pbi.13807) identified a TE insertion in _Brassica rapa_ _MYB18.1_. MYB18 is a positive regulator of the phyA photoresponse. A 4.8kb LTR/Copia disrupts the MYB gene. The comparison of a heading and a nonheading population of plants reveals almost complete dominance of this insertion allele in the heading population, while it was almost not present in the nonheading population. This suggest that _MYB18.1_ was under strong selection during domestication. The data also suggest that the insertion allele was later re-introduced into other cultivars through breeding.


### (73) _Oryza sativa OsFLP_ contributes to drought and salt tolerance ###
> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** OsFLP (MYB124) contributes to drought and salt stress response in rice through activation of OsNAC1 and OsNAC6 #MybMonday (details: https://lnk.tu-bs.de/oCdBQv)

<a href="https://twitter.com/boas_pucker/status/1546381272099979264">
<img alt="_Oryza sativa_ _OsFLP_ contributes to drought and salt tolerance (Tweet #73)" src="https://pbs.twimg.com/media/FXVPgDYXoAEQFnQ?format=jpg&name=900x900" width="40%" height="40%">
</a>

[Qu et al., 2022](https://doi.org/10.3390/ijms23115873) investigated _OsFLP_ which contributed to the drought and salt stress response of rice through activation of _OsNAC1_ and _OsNAC6_. FOUR LIPS (FLP) is classified as MYB124 in _Arabidopsis thaliana_. AtMYB124 and AtMYB88 have similar functions in regulating symmetric division of guard mother cells and abiotic stress response. 


### (74) _Musa_ R2R3-MYBs activate the anthocyanin biosynthesis ###
> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** Musa MYBs were identified as MBW components and activators of the anthocyanin biosynthesis. A complex with monocot bHLH ZmR resulted in higher target gene activation than a complex with dicot bHLH AtEGL3 #MybMonday (details: https://lnk.tu-bs.de/xrsVyn & https://doi.org/10.1101/2022.08.15.503939).

<a href="https://github.com/bpucker/MybMonday">
<img alt="_Musa_ R2R3-MYBs activate the anthocyanin biosynthesis (Tweet #74)" src="https://pbs.twimg.com/media/FS1t1heX0AIjqTi?format=jpg&name=large" width="50%" height="50%">
</a>

[Busche et al., 2022](https://doi.org/10.1101/2022.08.15.503939) identified three R2R3-MYBs that are involved in the formation of MBW complexes and the activation of the anthocyanin biosynthesis in _Musa_. The MBW complex is named after the three components: MYB-bHLH-WD40. The MYBs MusaMYB-Î±, MusaMYB-Î², and MusaMYB-Î³ showed a stronger activation of their target genes when forming a complex with the monocot bHLH ZmR compared to complexes with the dicot bHLH AtEGL3. This supports a systematic difference in the MYB-bHLH interaction between monocots and dicots.


### (75) Evolution of flavonol regulators in Brassicaceae II ###
> **_<a href="https://twitter.com/boas_pucker/status/1564138278919471105">TWEET</a>:_** Duplication of MYB21 and loss of the close paralog MYB24 seem to coincide in the evolution of Brassicacea just prior to emergence of the Brassiceae #MybMonday (details: https://lnk.tu-bs.de/oS4r2V & https://doi.org/10.1186/s12864-022-08819-8)

<a href="https://twitter.com/boas_pucker/status/1564138278919471105">
<img alt="Evolution of flavonol regulators in Brassicaceae II (Tweet #75)" src="https://pbs.twimg.com/media/Fav7r0_XgAE05no?format=jpg&name=large" width="50%" height="50%">
</a>

[Schilbert & Glover, 2022](https://doi.org/10.1186/s12864-022-08819-8) performed a comprehensive analysis of the MYB gene family in the Brassicaceae. The authors report that _MYB21_ was duplicated prior to the emergence of Brassiceae. The event seems to coincide with the loss of _MYB24_ which is a close paralog of _MYB21_. It is expected that the remaining paralogs will compensate for the _MYB24_ loss.


### (76) Proanthocyanidin biosynthesis regulators in chickpea ###
> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** Genome-wide identification of MYBs in chickpea revealed two regulators of the proanthocyanidin biosynthesis which are promising targets for crop improvement #MybMonday details: https://lnk.tu-bs.de/JYRVLZ.


[Rajput et al., 2022](https://doi.org/10.1007/s00425-022-03979-z) identified 119 typical MYBs in _Cicer arietinum_ (chickpea). CaPAR1 and CaPAR2 were identified as regulators of the proanthocyanidin biosynthesis. They interact with the bHLH transcription factor CaTT8. Cotransfection studies with CaPAR, CaTT8, and CaTTG1 show activation of the proanthocyanidin biosynthesis genes CaLAR and CaANR. The role of LAR and ANR in the context of the flavonoid biosynthesis is displayed in a [simplified illustration of the metabolic network](https://doi.org/10.3390/plants9091103). As CaPAR1 and CaPAR2 activate the proanthocyanidin biosynthesis, they are promising targets for future crop improvements. Engineering of the flavonoid biosynthesis is of economical relevance.




### MybMonday posts will become less frequent ###
Due to a lack of recent open access publications about the MYB transcription factors, the frequency of posts will decrease. However, we are happy to share high quality findings once they get published.



## Contact ##
Do you have a recent publication about MYBs in plants that could be featured here? Please get in touch: [Boas Pucker (email)](mailto:b.pucker@tu-braunschweig.de?subject=[GitHub]MYB_Monday)


## References ##

<details>
<summary>(click to expand)</summary>
<p>

<b>General introduction</b>:

Pucker B., Pandey A., Weisshaar B. and Stracke R. (2020). The R2R3-MYB gene family in banana (<i>Musa acuminata</i>): genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. doi: [10.1371/journal.pone.0239275](https://doi.org/10.1371/journal.pone.0239275).

Thiedig K, Weisshaar B, Stracke R (2021). Functional and evolutionary analysis of the Arabidopsis 4R-MYB protein SNAPc4 as part of the SNAP complex. Plant Physiology. doi: [10.1093/plphys/kiaa067](https://pub.uni-bielefeld.de/record/2949074).

Dubos C, Stracke R, Grotewold E, Weisshaar B, Martin C, Lepiniec L (2010). MYB transcription factors in Arabidopsis. Trends in Plant Science 15(10): 573-581. doi: [10.1016/j.tplants.2010.06.005 ](https://pubmed.ncbi.nlm.nih.gov/20674465/).

  
<b>References per post:</b>

1: Pucker B., Pandey A., Weisshaar B. and Stracke R. (2020). The R2R3-MYB gene family in banana (<i>Musa acuminata</i>): genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. doi: [10.1371/journal.pone.0239275](https://doi.org/10.1371/journal.pone.0239275).

2: Dubos C, Stracke R, Grotewold E, Weisshaar B, Martin C, Lepiniec L (2010). MYB transcription factors in Arabidopsis. Trends in Plant Science 15(10): 573-581. doi: [10.1016/j.tplants.2010.06.005](https://pubmed.ncbi.nlm.nih.gov/20674465/).
  
3: Baudry A, Heim MA, Dubreucq B, Caboche M, Weisshaar B, Lepiniec L (2004). TT2, TT8, and TTG1 synergistically specify the expression of BANYULS and proanthocyanidin biosynthesis in Arabidopsis thaliana. doi: [10.1111/j.1365-313X.2004.02138.x](https://pubmed.ncbi.nlm.nih.gov/15255866/).

4: ChagnÃ© D, Lin-Wang K, Espley RV, et al. An ancient duplication of apple MYB transcription factors is responsible for novel red fruit-flesh phenotypes. Plant Physiol. 2013;161(1):225-239. doi: [10.1104/pp.112.206771](https://dx.doi.org/10.1104%2Fpp.112.206771)
  
5: Jin, W., Wang, H., Li, M., Wang, J., Yang, Y., Zhang, X., Yan, G., Zhang, H., Liu, J. and Zhang, K. (2016), The R2R3 MYB transcription factor PavMYB10.1 involves in anthocyanin biosynthesis and determines fruit skin colour in sweet cherry (Prunus avium L.). Plant Biotechnol J, 14: 2120-2133. doi: [10.1111/pbi.12568](https://doi.org/10.1111/pbi.12568)
 
6:  Wang, H., Zhang, H., Yang, Y., Li, M., Zhang, Y., Liu, J., Dong, J., Li, J., Butelli, E., Xue, Z., Wang, A., Wang, G., Martin, C.and Jin, W. (2020) The control of red colour by a family of MYB transcription factors in octoploid strawberry (Fragaria Ã— ananassa) fruits. Plant Biotechnol J., https://doi.org/10.1111/pbi.13282
  
7: Fraser, L.G., Seal, A.G., Montefiori, M. et al. An R2R3 MYB transcription factor determines red petal colour in an Actinidia(kiwifruit) hybrid population. BMC Genomics 14, 28 (2013). https://doi.org/10.1186/1471-2164-14-28

8: Xi, W., Feng, J., Liu, Y. et al. The R2R3-MYB transcription factor PaMYB10 is involved in anthocyanin biosynthesis in apricots and determines red blushed skin. BMC Plant Biol 19, 287 (2019). https://doi.org/10.1186/s12870-019-1898-4
  
9: Hatlestad, G., Akhavan, N., Sunnadeniya, R. et al. The beet Y locus encodes an anthocyanin MYB-like protein that activates the betalain red pigment pathway. Nat Genet 47, 92â€“96 (2015). https://doi.org/10.1038/ng.3163

  Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., Pucker, B., Lopez-Nieves, S., Guo, R. and Brockington, S. (2019), The evolution of betalain biosynthesis in Caryophyllales. New Phytol, 224: 71-85. https://doi.org/10.1111/nph.15980
  
10: Stracke, R., HoltgrÃ¤we, D., Schneider, J. et al. Genome-wide identification and characterisation of R2R3-MYB genes in sugar beet (Beta vulgaris). BMC Plant Biol 14, 249 (2014). https://doi.org/10.1186/s12870-014-0249-8
  
11: Lee M M & Schiefelbein J. et al., 1999. WEREWOLF, a MYB-Related Protein in Arabidopsis, Is a Position-Dependent Regulator of Epidermal Cell Patterning. Cell Volume 99, Issue 5, 24 November 1999, Pages 473-483.

Baihui Wang, Qiang Luo, Yingping Li, Liufan Yin, Nana Zhou, Xiangnan Li, Jianhua Gan, Aiwu Dong, Structural insights into target DNA recognition by R2R3-MYB transcription factors, Nucleic Acids Research, Volume 48, Issue 1, 10 January 2020, Pages 460â€“471, https://doi.org/10.1093/nar/gkz1081
  
12: Zhu S, Shi W, Jie Y, Zhou Q, Song C (2020) A MYB transcription factor, BnMYB2, cloned from ramie (Boehmeria nivea) is involved in cadmium tolerance and accumulation. PLOS ONE 15(5): e0233375. https://doi.org/10.1371/journal.pone.0233375
  
13: MuÃ±oz-GÃ³mez S, SuÃ¡rez-Baron H, Alzate JF, GonzÃ¡lez F and PabÃ³n-Mora N (2021) Evolution of the Subgroup 6 R2R3-MYB Genes and Their Contribution to Floral Color in the Perianth-Bearing Piperales. Front. Plant Sci. 12:633227. doi: 10.3389/fpls.2021.633227
  
14: Rodrigues, J.A., Espley, R.V. & Allan, A.C. Genomic analysis uncovers functional variation in the C-terminus of anthocyanin-activating MYB transcription factors. Hortic Res 8, 77 (2021). https://doi.org/10.1038/s41438-021-00514-1

Stracke R, Werber M, Weisshaar B. The R2R3-MYB gene family in Arabidopsis thaliana. Curr Opin Plant Biol. 2001 Oct;4(5):447-56. doi: 10.1016/s1369-5266(00)00199-0. PMID: 11597504.

15: Alex Harkess, Kun Huang, Ron van der Hulst, Bart Tissen, Jeffrey L. Caplan, Aakash Koppula, Mona Batish, Blake C. Meyers, Jim Leebens-Mack, Sex Determination by Two Y-Linked Genes in Garden Asparagus, The Plant Cell, Volume 32, Issue 6, June 2020, Pages 1790â€“1796, https://doi.org/10.1105/tpc.19.00859
  
16: Peng, Y., Thrimawithana, A.H., Cooney, J.M. et al. The proanthocyanin-related transcription factors MYBC1 and WRKY44 regulate branch points in the kiwifruit anthocyanin pathway. Sci Rep 10, 14161 (2020). https://doi.org/10.1038/s41598-020-70977-0
  
17: Chen-Kun Jiang, Guang-Yuan Rao, Insights into the Diversification and Evolution of R2R3-MYB Transcription Factors in Plants, Plant Physiology, Volume 183, Issue 2, June 2020, Pages 637â€“655, https://doi.org/10.1104/pp.19.01082

18: Frerigmann H and Gigolashvili T (2014) Update on the role of R2R3-MYBs in the regulation of glucosinolates upon sulfur deficiency. Front. Plant Sci. 5:626. doi: 10.3389/fpls.2014.00626

19: Cheng J, Yu K, Shi Y, Wang J and Duan C (2021) Transcription Factor VviMYB86 Oppositely Regulates Proanthocyanidin and Anthocyanin Biosynthesis in Grape Berries. Front. Plant Sci. 11:613677. doi: 10.3389/fpls.2020.613677
  
20: Koo, Y., Poethig, R.S. Expression pattern analysis of three R2R3-MYB transcription factors for the production of anthocyanin in different vegetative stages of Arabidopsis leaves. Appl Biol Chem 64, 5 (2021). https://doi.org/10.1186/s13765-020-00584-0
  
21:  Genetic basis and dual adaptive role of floral pigmentation in sunflowers. Marco Todesco, Natalia Bercovich, Amy Kim, Ivana Imerovski, Gregory L. Owens, Ã“scar Dorado Ruiz, Srinidhi V. Holalu, Lufiani L. Madilao, Mojtaba Jahani, Jean-SÃ©bastien LÃ©garÃ©, Benjamin K. Blackman, Loren H. Rieseberg. bioRxiv 2021.06.26.449999; doi: https://doi.org/10.1101/2021.06.26.449999 
  
22: Wang J, Liu Y, Tang B, Dai X, Xie L, Liu F and Zou X (2020) Genome-Wide Identification and Capsaicinoid Biosynthesis-Related Expression Analysis of the R2R3-MYB Gene Family in Capsicum annuum L. Front. Genet. 11:598183. doi: 10.3389/fgene.2020.598183
  
23: Cao Y, Jia H, Xing M, Jin R, Grierson D, Gao Z, Sun C, Chen K, Xu C and Li X (2021) Genome-Wide Analysis of MYB Gene Family in Chinese Bayberry (Morella rubra) and Identification of Members Regulating Flavonoid Biosynthesis. Front. Plant Sci. 12:691384. doi: 10.3389/fpls.2021.691384
  
24: Xingyu Zheng, Kuenzang Om, Kimmy A Stanton, Daniel Thomas, Philip A Cheng, Allison Eggert, Emily Simmons, Yao-Wu Yuan, Gregory D Conradi Smith, Joshua R Puzey, Arielle M Cooley, The regulatory network for petal anthocyanin pigmentation is shaped by the MYB5a/NEGAN transcription factor in Mimulus, Genetics, Volume 217, Issue 2, February 2021, iyaa036, https://doi.org/10.1093/genetics/iyaa036

Ding B, Patterson EL, Holalu SV, Li J, Johnson GA, Stanley LE, Greenlee AB, Peng F, Bradshaw HD Jr, Blinov ML, Blackman BK, Yuan YW. Two MYB Proteins in a Self-Organizing Activator-Inhibitor System Produce Spotted Pigmentation Patterns. Curr Biol. 2020 Mar 9;30(5):802-814.e8. doi: 10.1016/j.cub.2019.12.067. Epub 2020 Feb 20. PMID: 32155414; PMCID: PMC7156294.

25:  Systematic analysis of the R2R3-MYB family of transcription factors in Camellia sinensis: evidence for species-specific catechin biosynthesis regulation. Jingyi Li, Shaoqun Liu, Peifen Chen, Jiarong Cai, Song Tang, Wei Yang, Fanrong Cao, Peng Zheng, Binmei Sun. bioRxiv 2021.07.21.453189; doi: https://doi.org/10.1101/2021.07.21.453189 
  
26: Yu, J., Zhao, G., Li, W. et al. A single nucleotide polymorphism in an R2R3 MYB transcription factor gene triggers the male sterility in soybean ms6 (Ames1). Theor Appl Genet 134, 3661â€“3674 (2021). https://doi.org/10.1007/s00122-021-03920-0
  
27: Feng G, Burleigh JG, Braun EL, Mei W, Barbazuk WB. Evolution of the 3R-MYB Gene Family in Plants. Genome Biol Evol. 2017 Apr 1;9(4):1013-1029. doi: 10.1093/gbe/evx056. PMID: 28444194; PMCID: PMC5405339.
  
28: MYB quiz: https://colab.research.google.com/drive/1gBvRBkO-aGf3VJWkKc3aG-oVuBo80D2z?usp=sharing
  
29: Pucker B, Pandey A, Weisshaar B, Stracke R (2020) The R2R3-MYB gene family in banana (Musa acuminata): Genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. https://doi.org/10.1371/journal.pone.0239275
  
30:  Genome-wide characterization of the R2R3-MYB transcription factors in pepper (Capsicum spp.) unveils the role of CaMYB101 as repressor in anthocyanin biosynthesis. Ying Liu, Yi Wu, Zicheng Wang, Shiya Zhang, Xintong Liu, Yury Tikunov, Richard G.F. Visser, Rob E. Schouten, Leo F.M. Marcelis, Zhao Zhang, Arnaud Bovy. bioRxiv 2021.08.24.457473; doi: https://doi.org/10.1101/2021.08.24.457473 
  
31: Stracke, R., HoltgrÃ¤we, D., Schneider, J. et al. Genome-wide identification and characterisation of R2R3-MYB genes in sugar beet (Beta vulgaris). BMC Plant Biol 14, 249 (2014). https://doi.org/10.1186/s12870-014-0249-8

Timoneda, A., Feng, T., Sheehan, H., Walker-Hale, N., Pucker, B., Lopez-Nieves, S., Guo, R. and Brockington, S. (2019), The evolution of betalain biosynthesis in Caryophyllales. New Phytol, 224: 71-85. https://doi.org/10.1111/nph.15980
  
32: Pucker B, Pandey A, Weisshaar B, Stracke R (2020) The R2R3-MYB gene family in banana (Musa acuminata): Genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. https://doi.org/10.1371/journal.pone.0239275
  
33: An R2R3 MYB transcription factor confers brown planthopper resistance by regulating the phenylalanine ammonia-lyase pathway in rice. Jun He, Yuqiang Liu, Dingyang Yuan, Meijuan Duan, Yanling Liu, Zijie Shen, Chunyan Yang, Zeyu Qiu, Daoming Liu, Peizheng Wen, Jie Huang, Dejia Fan, Shizhuo Xiao, Yeyun Xin, Xianian Chen, Ling Jiang, Haiyang Wang, Longping Yuan, Jianmin Wan. Proceedings of the National Academy of Sciences Jan 2020, 117 (1) 271-277; DOI: 10.1073/pnas.1902771116
  
34:  Major orchestration of shikimate, early phenylpropanoid and stilbenoid pathways by Subgroup 2 R2R3-MYBs in grapevine. Luis OrduÃ±a, Miaomiao Li, David Navarro-PayÃ¡, Chen Zhang, Antonio Santiago, Pablo Romero, Å½iva RamÅ¡ak, Gabriele Magon, Janine HÃ¶ll, Patrik Merz, Kristina Gruden, Alessandro Vannozzi, Dario Cantu, Jochen Bogs, Darren C. J. Wong, Shao-shan Carol Huang, JosÃ© TomÃ¡s Matus. bioRxiv 2020.12.31.424746; doi: https://doi.org/10.1101/2020.12.31.424746 
  
35: Xueying Zhang, Yuqing He, Linying Li, Hongru Liu, Gaojie Hong, Involvement of the R2R3-MYB transcription factor MYB21 and its homologs in regulating flavonol accumulation in Arabidopsis stamen, Journal of Experimental Botany, Volume 72, Issue 12, 28 May 2021, Pages 4319â€“4332, https://doi.org/10.1093/jxb/erab156
  
36: Kenta Shirasawa, Sogo Nishio, Shingo Terakami, Roberto Botta, Daniela Torello Marinoni, Sachiko Isobe, Chromosome-level genome assembly of Japanese chestnut (Castanea crenata Sieb. et Zucc.) reveals conserved chromosomal segments in woody rosids, DNA Research, Volume 28, Issue 5, October 2021, dsab016, https://doi.org/10.1093/dnares/dsab016
  
Automatic identification and annotation of MYB gene family members in plants. Boas Pucker. bioRxiv 2021.10.16.464636; doi: https://doi.org/10.1101/2021.10.16.464636 
  
37: Declan J Lafferty, Richard V Espley, Cecilia H Deng, Catrin S GÃ¼nther, Blue Plunkett, Janice L Turner, Laura Jaakola, Katja Karppinen, Andrew C Allan, Nick W Albert, Hierarchical regulation of MYBPA1 by anthocyanin- and proanthocyanidin-related MYB proteins is conserved in Vaccinium species, Journal of Experimental Botany, 2021;, erab460, https://doi.org/10.1093/jxb/erab460
  
38: Li Y, Chen X, Wang J, Zou G, Wang L, et al. (2020) Two responses to MeJA induction of R2R3-MYB transcription factors regulate flavonoid accumulation in Glycyrrhiza uralensis Fisch. PLOS ONE 15(7): e0236565. https://doi.org/10.1371/journal.pone.0236565
  
39: Zhang D, Tan Y, Dong F, Zhang Y, Huang Y, Zhou Y, Zhao Z, Yin Q, Xie X, Gao X, Zhang C and Tu N (2021) The Expression of IbMYB1 Is Essential to Maintain the Purple Color of Leaf and Storage Root in Sweet Potato [Ipomoea batatas (L.) Lam]. Front. Plant Sci. 12:688707. doi: 10.3389/fpls.2021.688707
  
40: Automatic identification and annotation of MYB gene family members in plants. Boas Pucker. bioRxiv 2021.10.16.464636; doi: https://doi.org/10.1101/2021.10.16.464636 
  
41: Thiedig K, Weisshaar B, Stracke R (2021). Functional and evolutionary analysis of the Arabidopsis 4R-MYB protein SNAPc4 as part of the SNAP complex. Plant Physiology. doi: [10.1093/plphys/kiaa067](https://pub.uni-bielefeld.de/record/2949074).

  
  
  
  </p>
</details>
