# #MybMonday
This repository provides additional background to the #MybMonday tweets

## Introduction ##

The MYB transcription factors from one of the largest transcription factor families in plants. The characteristic feature of MYBs are repeat units which harbour conserved tryptophan (W) residues with a defined number of interleaved amino acid residues ([Dubos et al., 2010](https://doi.org/10.1016/j.tplants.2010.06.005)). These repeat units determine the names of MYBs. For example, the 'R2R3 MYBs' harbour two MYB repeat units: the R2 and the R3 repeat. Especially the R2R3 clade of the MYB family is expanded in plants. Members of this family are phylogenetically assigned to groups. It is expected that members of a certain group control a specific function or process. Therefore, it is possible to transfer the knowledge about MYBs studied in one plant species to the MYBs identified in a new species. Due to the large number of MYBs, they are involved in numerous processes.

Here is a list of the expected functions of [MYBs in banana (_Musa acuminata_)](https://doi.org/10.1371/journal.pone.0239275): (1) defense/stress response, (2) trichome branching and petal morphogenesis, (3) defense/stress response, (4) glucosinolates/camalexin, (5) suberin, (6) root development, (7) phenylpropanoid/lignin, (8) flower meristem identity, (10) abiotic stress response, flower morphogenesis, and stilbene biosynthesis, (11) photomorphogenesis, (12) anther and tapetum development, (14) axillary meristem and root growth, (15) SCW and lignin, (16) stemen/anther development, phenylpropanoid, SCW, (17) anther and trichome development, (18) SCW and lignin, (20) mucillage, lignin, and stomatal closure, (22) repressors of phenylpropanoid, sinapata, lignin, (23) general flavonoid biosynthesis and trichome development, (24) anthocyanins (monocots), (25) proanthocyanins, (27) flavonoid repressors, (28) trichome development, roothair development, floral organ development, (29) anthocyanins and betalains (dicots), (30) flavonols and phlobaphene, (31) cell cycle regulation, (33) phenylpropanoid and lignin, (34) stamen development, (35) defense and stress response, (36) leaf, shoot, germ morphogenesis, (37) anther development and stress response, (38) embryogenesis and seed maturation, (39) guard cell division and root gravitropism, (40) stress response and hormone signaling, (41) stress tolerance, and (42) cell wall, lignin, seed oil,axillary meristem. The phylogenetic relationship of these MYBs is displayed in Fig. 1.


<a href="https://doi.org/10.1371/journal.pone.0239275">
<img alt="Phylogenetic tree of banana MYBs" src="https://www.biorxiv.org/content/biorxiv/early/2020/02/03/2020.02.03.932046/F2.large.jpg" width="75%" height="75%">
</a>

Fig. 1: Phylotenetic tree of the banana (_Musa acuminata_) MYBs ([Pucker et al., 2020](https://doi.org/10.1371/journal.pone.0239275)).


In addition to the R2R3 MYBs, there are also [MYB3R (cell cycle control)](https://dev.biologists.org/content/134/6/1101), [MYB4R (SNAP complex)](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjJnMGJ6vruAhWLyaQKHc3rAysQFjABegQIBBAD&url=https%3A%2F%2Facademic.oup.com%2Fplphys%2Fadvance-article-pdf%2Fdoi%2F10.1093%2Fplphys%2Fkiaa067%2F35063342%2Fkiaa067.pdf&usg=AOvVaw04BwHFjy1QcAHg9-k5KeOO), and various others. However, these lineages have not expanded substantially in plants.



## Tweets with additional background ##

### (1) MYBs control diverse functions in plants ###

> **_<a href="https://twitter.com/boas_pucker/status/1363737787782995970">TWEET</a>:_** MYBs are among the largest transcription factor families in plants. Members of the R2R3-MYB clade control various processes including ontogenesis, cell wall biosynthesis, stress response, and specialized metabolism. #MybMonday (src: https://bit.ly/2NkoaYZ)

<a href="https://twitter.com/boas_pucker/status/1363737787782995970">
<img alt="Phylogenetic tree of banana MYBs (Tweet #1)" src="https://pbs.twimg.com/media/EuzsUIJXcAEPIpy?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


This tweet serves as an introduction. Please read the introduction section on top of this page for additional information. The phylogenetic tree shows the diversity of different functions that are regulated by MYB transcription factors. It appears that there are cases of convergent evolution i.e. different clades of MYBs regulating the same process.


### (2) MYB repeat structure ###

> **_<a href="https://twitter.com/boas_pucker/status/1363858585768263686">TWEET</a>:_** Characteristic feature of MYBs are conserved tryptophan residues at defined positions in MYB repeat units. Names of MYBs are based on these MYB repeats e.g. 'R2R3-MYBs' harbour one R2 and one R3 repeat. #MybMonday (src: https://bit.ly/2ZBfWOD)

<a href="https://twitter.com/boas_pucker/status/1363858585768263686">
<img alt="MYB repeat structure (Tweet #2)" src="https://pbs.twimg.com/media/EuzuOVBWQAAfvEM?format=png&name=medium" width="50%" height="50%">
</a>


These characteristic repeat features can be harnessed for the identification of MYBs in transcriptome or genome assemblies. Regular expressions can be used to search for this amino acid pattern in a given set of peptide sequences. Many genome-wide studies of R2R3-MYBs use the presence of this conserved domain to select their candidates.


### (3) MYBs interact with bHLH and WD40 (MBW complex) ###

> **_<a href="https://twitter.com/boas_pucker/status/1366274503207227392">TWEET</a>:_** Some MYBs form complexes with bHLH and WD40 proteins. These complexes are named MBW after the 3 involved TFs. MBW controls the seed coat colour in Arabidopsis thaliana and other developmental processes. #MybMonday (src: https://bit.ly/2ZCoswD)

<a href="https://twitter.com/boas_pucker/status/1366274503207227392">
<img alt="MYBs interact with bHLH and WD40 (Tweet #3)" src="https://pbs.twimg.com/media/EvNNzvcXIAEq1tv?format=jpg&name=medium" width="30%" height="30%">
</a>


Different MYBs and bHLHs can be involved in the MBW complex formation. The integration of different TFs allows specificity for different target genes. This complex is not only required for the regulation of the proanthocyanidin biosynthesis, but also plays a role in the development of trichomes and root hairs.

### (4) MYBs are breeding targets ###

> **_<a href="https://twitter.com/boas_pucker/status/1368811218014629888">TWEET</a>:_** MYBs are the target of breeding projects to improve fruits, vegetables, and ornamental plants. MYB1/10/110a are responsible for the red coloration of apple skin, foliage & fruit-flesh, respectively, by activating a pigment biosynthesis  #MybMonday (src: https://bit.ly/37OrRgb)

<a href="https://twitter.com/boas_pucker/status/1368811218014629888">
<img alt="MYBs are breeding targets (Tweet #4)" src="https://pbs.twimg.com/media/EvNPR_CXMAEvIFp?format=jpg&name=medium" width="30%" height="30%">
</a>


As transcriptional regulators of entire biosnythesis pathways, MYBs are promising targets of breeding projects. The deactivation of a specific MYB and be used to shut down a pathway leading to undesired products. Additionally, the contruction of genetically modified plants, e.g. overexpressing certain MYBs, is an option to improve the crop quality by up-regulating desired pathways.


### (5) MYB associated with sweet cherry colour ###

> **_<a href="https://twitter.com/boas_pucker/status/1371347932851634177">TWEET</a>:_** PavMYB10.1 is associated with the fruit skin colour of sweet cherry. Three different alleles lead to yellow to red colouration in the varieties 'Big Dragon', 'Rainier', and 'Lapins'. #MybMonday (src: https://bit.ly/3swLEsx)

<a href="https://twitter.com/boas_pucker/status/1371347932851634177">
<img alt="MYB associated with sweet cherry colour (Tweet #5)" src="https://pbs.twimg.com/media/EvNSbz7XMAAZqcG?format=jpg&name=medium" width="30%" height="30%">
</a>


### (6) Red and white strawberries ###

> **_<a href="https://twitter.com/boas_pucker/status/1373884648485445633">TWEET</a>:_** The red colour of octoploid strawberry (Fragaria × ananassa) is controlled by a MYB. A 8bp insertion in the CDS of FaMYB10 can change the colour from red to white through a single loss-of-function allele. #MybMonday (src: https://bit.ly/2ZUIGBV)

<a href="https://twitter.com/boas_pucker/status/1373884648485445633">
<img alt="Red and white strawberries (Tweet #6)" src="https://pbs.twimg.com/media/EvNUujRWgAIgh6r?format=jpg&name=medium" width="30%" height="30%">
</a>


This loss-of-function allele of MYB10-2 needs to be present in a homozygous state to cause the white phenotype as it is not dominant negative. Interesting about the MYB10-2 is the lack of introns wich suggest that it originates from a reverse transcription event. It will be exciting to completely resolve the story about the anthocyanins regulating MYBs in various strawberry strains. It is also a remarkable example of pathway inactivation through loss of a regulator. This is frequently observed in the flavonoid biosynthesis. One reason could be the involvement of structural genes in multiple branches of the pathway. The loss of the anthocyanin specific regulator only inactivaites one function, while loss of the structural gene would break multiple branches.


### (7) Colouration of kiwi fruit tissues ###

> **_<a href="https://twitter.com/boas_pucker/status/1376406263693783042">TWEET</a>:_** The red colour of kiwifruit petals is determined by MYB110a, but there might be more MYBs involved in the anthocyanin biosynthesis regulation in other tissues. A microsatellite marker linked to MYB110a revealed the location on chr10. #MybMonday (src: https://bit.ly/3aVRWvZ)

<a href="https://twitter.com/boas_pucker/status/1376406263693783042">
<img alt="Colouration of kiwi fruit tissues (Tweet #7)" src="https://pbs.twimg.com/media/EvNXel3XcAAW_75?format=jpg&name=large" width="50%" height="50%">
</a>


### (8) Red blushed skin of apricots ###

> **_<a href="https://twitter.com/boas_pucker/status/1378942978253807616">TWEET</a>:_** Consumers prefer apricots with red blushed skin. Expression of PaMYB10 correlates with expression of anthocyanin biosynthesis genes which are responsible for production of red pigments. PaMYB10 seems to be an important activator. #MybMonday (src: https://bit.ly/2ZXbWbo)

<a href="https://twitter.com/boas_pucker/status/1378942978253807616">
<img alt="Red blushed skin of apricots (Tweet #8)" src="https://pbs.twimg.com/media/EvNbhhvXYAE5-2R?format=png&name=900x900" width="30%" height="30%">
</a>


### (9) Betalain biosynthesis in sugar beet is controlled by BvMYB1 ###

> **_<a href="https://twitter.com/boas_pucker/status/1381479694483156992">TWEET</a>:_** BvMYB1 is responsible for the red colouration of sugar beets and beetroots. This MYB activates the betalain biosynthesis which produces red pigments. #MybMonday (src: https://bit.ly/3q0w2fa)

<a href="https://twitter.com/boas_pucker/status/1381479694483156992">
<img alt="Betalain biosynthesis in sugar beet is controlled by BvMYB1 (Tweet #9)" src="https://pbs.twimg.com/media/EvNeVhRXAAUsYb0?format=png&name=900x900" width="30%" height="30%">
</a>

Anthocyanin and betalains are (to some extent) functionally redundant. Not a single plant species was detected to show pigmentation by anthocyanins and betalains which lead to the mutual exclusion theory ([Timoneda et al., 2019](https://doi.org/10.1111/nph.15980)). BvMYB1 is an ortholog of the anthocyanin regulator PAP1. The co-option of this anthocyanin regulator by the betalain biosynthesis makes the complex pigment evolution in the Caryophyllales even more exciting. In contrast to PAP1, BvMYB1 does not require a bHLH and TTG1 cofactor to activate the betalain biosynthesis. The loss of an anthocyanin regulator to the betalain biosynthesis could be one explanation why anthocyanins have been replaced by betalains in sugar beet.


### (10) Flavonol regulator BvMYB12 ###

> **_<a href="https://twitter.com/boas_pucker/status/1384016409202679808">TWEET</a>:_** MYB functions are not restricted to fruit colours. BvMYB12 activates flavonol biosynthesis genes in sugar beet independent of bHLH/WD40. Flavonols protect plants against UV light and ROS, but also regulate developmental processes. #MybMonday (src: https://bit.ly/3dSMgVo)


<a href="https://twitter.com/boas_pucker/status/1384016409202679808">
<img alt="Flavonol regulator BvMYB12 (Tweet #10)" src="https://pbs.twimg.com/media/EvNku2gWYAEWAeN?format=jpg&name=medium" width="40%" height="40%">
</a>

While betalains replace anthocyanins in many Caryophyllales families ([Timoneda et al., 2019](https://doi.org/10.1111/nph.15980)), flavonols are still present in all Caryophyllales lineages. The flavonol biosynthesis is an evolutionary old pathway which appears to be conserved across all land plant lineages. The identificaiton of the AtMYB ortholog BvMYB12, which is also a flavonol biosynthesis regulator, indicates that the regulation of this pathway is also conserved over long phylogenetic distances.


### (11) Structure of WEREWOLF ###

> **_<a href="https://twitter.com/boas_pucker/status/1386553123402043392">TWEET</a>:_** Third alpha-helices of R2 and R3 repeats of WEREWOLF fit in the major groove of the DNA and recognize AACNGC. L59 of WER is probably sensing the methylation status of the binding site. #MybMonday (details: https://bit.ly/3nmpR5n)


<a href="https://twitter.com/boas_pucker/status/1386553123402043392">
<img alt="Structure of WEREWOLF (Tweet #11)" src="https://pbs.twimg.com/media/Ez2Ah4qXsAQKyCW?format=jpg&name=medium" width="40%" height="40%">
</a>

WEREWOLF (WER) is an important regulator in the plant ontogenesis [Lee & Schiefelbein, 1999](https://doi.org/10.1016/S0092-8674(00)81536-6). Generally, cis-elements of R2R3-MYBs can be assigned to the (C/T)AACNG or the ACC(A/T)A(A/C) group. A recent study by [Wang et al., 2019](https://academic.oup.com/nar/article/48/1/460/5626523) investigated the structure of WER and the interaction with cis-elements: The third alpha-helices of R2 and R3 repeats of WEREWOLF fit in the major groove of the DNA and recognize AACNGC. K55, N106, N109, and N110 are required for WER function. The L59 of WER is probably involved in sensig DNA methylation at 5mC. This 5mC methylation or a 6mA methylation in the AAC element might be able to prevent the interaction of WER and the DNA. L59 is a substitution in WER compared to MsMyb, TvMyb1, and TvMyb3. The L59A mutant shows almost the same affinity to methylated and unmethylated binding sites.


### (12) MYB2 of Boehmeria nivea ###

> **_<a href="https://twitter.com/boas_pucker/status/1389089839534968835">TWEET</a>:_** The R1-MYB family member MYB2 of Boehmeria nivea (Ramie) appears to be a positive regulator of Cd tolerance and accumulation. #MybMonday (details: https://bit.ly/3evEiQH)

<a href="https://twitter.com/boas_pucker/status/1389089839534968835">
<img alt="MYB2 of <i>Boehmeria nivea</i> (Tweet #12)" src="https://pbs.twimg.com/media/Ez2HyD-XEAApfcv?format=png&name=900x900" width="40%" height="40%">
</a>


A recent publication by [Zhu et al., 2020](https://doi.org/10.1371/journal.pone.0233375) describes MYB2 of Boehmeria nivea (Ramie) as regulator of Cadmium (Cd) tolerance and accumulation. BnMYB2 is an ortholog of GmMYB176, StMYB1R-1, AtMYBS3, OsMYBS1, and OsMYBS2. This clade is characterized by a conserved SHAQKYF motif. Heterologous expression of BnMYB2 in A. thaliana seem to increase the Cd tolerance and accumulation significantly. Would the expression of other MYBs e.g. AtMYBS3 result in a similar effect?


### (13) Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae ###

> **_<a href="https://twitter.com/boas_pucker/status/1391634107545829376">TWEET</a>:_** Subgroup 6 of the R2R3-MYBs controls the flavonoid biosynthesis which is responsible for the colouration of flowers. These MYBs contribute to the diversity of floral colour in Aristolochiaceae. #MybMonday (details:https://bit.ly/3y2PhdA)


<a href="https://twitter.com/boas_pucker/status/1391634107545829376">
<img alt="Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae (Tweet #13)" src="https://pbs.twimg.com/media/E1APsxnX0AQc5Af?format=jpg&name=medium" width="40%" height="40%">
</a>


Subgroup 6 R2R3-MYBs control floral color in Aristolochiaceae
[Munoz-Gomez et al.](https://doi.org/10.3389/fpls.2021.633227) report that the Aristolochiaceae are characterized by a huge diversity of flower coloration and forms. While the subgroup 6 of the R2R3-MYBs is generally responsible for regulation of the flavonoid biosynthesis (which includes colourful anthocyanins), there are several lineage-specific MYB gene duplications which cause alterations in the activity of regulated biosynthesis pathways. These TF differences result ultimately in the diversity of flower colors through different activation of pigment pathways. The authors investigated the expression of flavonoid biosynthesis genes in multiple species. No F3'5'H was detected in the transcriptome analyses which is in line with the absence of blue pigmented flowers. DFR and ANS were not detected in some of the datasets which would indicate that no anthocyanins are produced in the investigated tissues.


### (14) Conserved motifs in the C-terminal part of anthocyanin activators ###

> **_<a href="https://twitter.com/boas_pucker/status/1394178368149458946">TWEET</a>:_** Activators of the anthocyanin biosynthesis belong to subgroup 6 of the R2R3-MYB family. In addition to the R2R3 domain, there are three short motifs conserved among anthocyanin regulators #MybMonday (details: https://bit.ly/3eTQH28)

<a href="https://twitter.com/boas_pucker/status/1394178368149458946">
<img alt="Conserved motifs in the C-terminal part of anthocyanin activators (Tweet #14)" src="https://pbs.twimg.com/media/E1karEGWEAE1Cu-?format=jpg&name=large" width="40%" height="40%">
</a>


[Rodrigues et al., 2021](https://www.nature.com/articles/s41438-021-00514-1) identified functional variation in the C-terminal region of anthocyanin-activating MYBs. Molecular recognition features (MoRFs) are located in regions of intrinsic disorder of transcription factors. They are often short linear motifs (SLiMs) of 3-11 amino acid residues with evolutionary variability, but some consereved residues which determine specificity in interactions [Stracke et al., 2001](https://pubmed.ncbi.nlm.nih.gov/11597504/). Flexibility in these regions is considered an important factor in the evolution of signalling pathways by interacting co-factors. Multiple subgroups of the MYBs interact with bHLH transcription factors, but the involved motifs evolved convergently in the C-terminal region (subgroup 4 and subgroup 12) or in the N-terminal region (subgroups 5, 6, 15, 16). Anthocyanin regulating MYBs are located in subgroup 6. The C-terminal motif in this subgroup overlaps with the TAS4-siRNA81(-) target sequence. The PA and anthocyanin regulation subgroups 5 und 6 appear to be derived from the flavonoid regulator subgroup 7. However, early branching members in the subgroup 6 are not able to activate anthocyanins or lead only to a weak activation. Functionally characterized subgroup 6 members show conservation of three short motifs in their C-terminal region.


### (15) MYB involved in sex determination ###

> **_<a href="https://twitter.com/boas_pucker/status/1396707538373156864">TWEET</a>:_** The R2R3-MYB gene DEFECTIVE IN TAPETAL DEVELOPMENT AND FUNCTION (TDF1) is one of two sex determination genes in Asparagus officinalis. #MybMonday (details: https://bit.ly/3yHB6Ll)

<a href="https://twitter.com/boas_pucker/status/1396707538373156864">
<img alt="MYB involved in sex determination (Tweet #15)" src="https://pbs.twimg.com/media/E2H9ewXWYAAH912?format=jpg&name=medium" width="30%" height="30%">
</a>


Dioecy has evolved hundreds of times independently from hermaphrody in plants. [Harkess et al., 2020](http://www.plantcell.org/content/32/6/1790) present a MYB that is involved in sex determination in _Asparagus officinalis_. The X and Y chromosomes of A. officinalis evolved from an ancestral autosome pair during the shift from hermaphroditism to dioecy. A loss of recombination in a Y chromosomal region with genes supressing the female organogenesis (_SOFF_) and promoting male function (_TDF1_) can be caused by hemizygosity. Two Y-speficific genes appear sufficient for sex determination. The analysis of different mutant lines revealed a 1Mbp region on the Y-chromosome that is responsible for the sex determination. One of the genes in this region is the R2R3-MYB transcription factor _DEFECTIVE IN TAPETAL DEVELOPMENT AND FUNCTION_ (_TDF1_). Interestingly, this gene is not sex linked in all dioceious Asparagus species which indicates that this sex determination mechanism evolved within the genus. Sex chromosome evolution often leads to the accumulation of male genes on the Y chromosome and female genes on the X chromsome. One example of an X-linked gene is _WIP2/NTT_ which is specificly expressed in ovules.


### (16) MYBC1 in kiwifruit ###

> **_<a href="https://twitter.com/boas_pucker/status/1399261076877885443">TWEET</a>:_** Kiwifruit MYBC1 is an activator of the flavonoid biosynthesis genes F3'H and F3'5'H. Up-regulation F3'H/F3'5'H is discussed in context of increased proanthocyanidin content #MybMonday (details: https://bit.ly/3vBUJlH)


<a href="https://twitter.com/boas_pucker/status/1399261076877885443">
<img alt="MYBC1 in kiwifruit (Tweet #16)" src="https://pbs.twimg.com/media/E2ssKpOXoAIEqiW?format=jpg&name=large" width="30%" height="30%">
</a>



[Peng et al., 2020](https://www.nature.com/articles/s41598-020-70977-0) present MYBC1 and WRKY44 as activators of anthocyanin and proanthocyanidin biosynthesis. Both transcription factors appear to activate F3'H and F3'5'H as well as the proanthocyanidin biosnythesis genes. The authors present that an increased hydroxylation (via F3'H and F3'5'H) increases the flux into the proanthocyanidin biosynthesis which competes with the anthocyanin biosynthesis for substrate. Member of the MYB subgroup 6 are usually activating the anthocyanin biosynthesis. However, MYBC1 belongs to the subgroup 5. Peng et al. used RNA-Seq of differently pigmented kiwifruit cultivars to identify the underlying genes. Transient overexpression of two candidate genes, MYBC1 and WRKY44, in tobacco caused accumulation of the anthocyanin in patches. MYBC1 overexpression leads to cyanidin-based and delphinidin-based anthocyanin, while WRKY44 overexpression only caused the accumulation of cyanidin-based anthocyanins.
Overexpression in calli revealed differences between MYB110 (anthocyanin actiator) and MYBC1/WRKY44. While MYB110 activates the F3GT which is required for anthocyanin biosynthesis, MYBC1/WRKY44 actiate the F3'H/F3'5'H and proanthocyanidin biosynthesis genes. Since the MYBC1/WRKY44 overexpression results in higher proanthocyanidin content, the authors speculate that both TFs might be PA regulators rather than general activators of the flavonoid biosynthesis.



### (17) Evolution of the R2R3-MYBs in plants ###

> **_<a href="https://twitter.com/boas_pucker/status/1401811164737150977">TWEET</a>:_** Ancestral land plants already had 10 subfamilies of R2R3-MYBs. Orthologs of 7 subfamilies were detected in chlorophytes and charophycean algae. Subfamily VIII expanded in land plants. #MybMonday (details: https://bit.ly/3irG1tV)

<a href="https://twitter.com/boas_pucker/status/1401811164737150977">
<img alt="Evolution of the R2R3-MYBs in plants (Tweet #17)" src="https://pbs.twimg.com/media/E3Q59hoWEAA0RIj?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

A phylogenetic anaylysis by [Jiang & Rao, 2020](https://doi.org/10.1104/pp.19.01082) revealed 10 subfamilies of R2R3-MYBs in the ancestral land plants. Seven of these subfamilies have orthologs in the chlorophytes and charophycean algae. Different subfamilies of the MYB expanded in different plant lineages. Whole genome duplication alone cannot explain this pattern. Chlorophyta (<13) and streptophyte alga _Klebsormidium nitens_ (22) have small numbers of MYBs, while land plants have generally a larger number. About half of the land plant R2R3-MYBs fall into subfamily VIII, which is the main contributor to the expansion of R2R3-MYBs in land plants. This study supports the hypothesis that the R1R2R3-MYBs are not the result of R2R3-MYBs gaining an additional repeat.
The authors point out some issues when investigating the evolution of MYBs:
(1) R2R3-MYBs have been classified into 23-90 subgroups thus there is little consensus regarding their classification schemes. Therefore, Jiang & Rao propose a new classification scheme which better reflects the asymmetric evolution of R2R3-MYBs in land plants and is restricted to monophyletic groups.
(2) Phylogenetic anlysis are impacted by the quality of the annotations of studied species. The numbers of R2R3-MYBs might be underestimated in some species.


### (18) R2R3-MYBs regulate gucosinolate biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1404310128912785412">TWEET</a>:_** MYB28, MYB29, MYB34, MYB51, MYB76, and MYB122 control the glucosinolate biosynthesis. Sulfur deficiency reduces the gene expression of MYB29 and MYB76 through negative regulation by SLIM1. #MybMonday (details: https://bit.ly/2RWQNh6)

<a href="https://twitter.com/boas_pucker/status/1404310128912785412">
<img alt="R2R3-MYBs regulate gucosinolate biosynthesis (Tweet #18)" src="https://pbs.twimg.com/media/E30ZdesWEAMCm7a?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

Plants need a sophisticated regulatory system to balance the flux of sulfur into glucosinolate (GSL) biosynthesis and central metabolism. MYB34, MYB51, and MYB122 control the indolic GSL (IG) biosynthesis, while MYB28, MYB29, and MYB76 control the aliphatic GSL (AG) biosynthesis. These MYBs also enhance sulfur assimilation through activation of the respective genes. IG molecules contain 2 sulfur atoms in their backbone, while AG molecules contain 3 sulfur atoms in their backbone. [Frerigmann & Gigolashvili, 2014](https://doi.org/10.3389/fpls.2014.00626) studied the role of R2R3-MYBs under sulfur deficiency. Previous studies had identified that sulfur assimilation pathway genes are up-regulated under sulfur deficiency and that GSL biosynthesis is reduced. Catabolism of GSL is one solution how sulfur can be made available in the cell again.

Sulfur Limitation 1 (SLIM1) was identified as a negative transcriptional regulator of MYB29 and MYB76. SLIM1 reduces the activities of both promoters in a trans-activation assay. Sulfur deficiency resulted in a reduction of the AG content. The biosynthesis of AGs might require a tighter regulation than the IG biosnythesis, because the AG molecules contain one additional sulfur atom compared to IGs. However, a 'low GSL signal' seems to prevent the SLIM1-based repression of the MYB29 and MYB76 expression. This was investigated in mutant plants with a block in the GSL biosynthesis. The authors speculate that a complete down-regulation of the MYBs is not possible, because the six MYBs are activating sulfur assimilarion genes in addition to the GLS biosnythesis. There are multiple scenarios which could explain the observed results and further research on the topic is required.


### (19) VvMYB86 - activator of PA and repressor of anthocyanin biosynthesis ###

> **_<a href="https://twitter.com/boas_pucker/status/1406846844878168064">TWEET</a>:_** Vitis vinifera MYB86 seems to be a positive regulator of the proanthocyanidin biosynthesis (LAR), but a negative regulator of the anthocyanin biosynthesis (ANS & UFGT) #MybMonday (details: https://bit.ly/3xuWods)


<a href="https://twitter.com/boas_pucker/status/1406846844878168064">
<img alt="VvMYB86 - activator of PA and repressor of anthocyanin biosynthesis (Tweet #19)" src="https://pbs.twimg.com/media/E4W9KcqXwAUq9KV?format=jpg&name=4096x4096" width="30%" height="30%">
</a>

[Cheng et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2020.613677/full) characterized the Vitis vinifera MYB86 through in vivo and in vitro experiments. It turns out that this MYB is a positive regulator of LAR, which is involved in the anthocyanin biosynthesis. LAR and ANR are the two enzymes of the PA biosynthesis. While ANR determines the total amount of proanthocyanidins, LAR determines the degree of PA polymerisation. While upregulating LAR, this VvMYB86 represses ANS and UFGT, which are involved in anthocyanin formation. Most previously characterized V. vinifera MYBs are dedicated regulators of one of the pathways. However, MYB5 in V. vinifera and additional MYBs in other species have been identified as regulators of both pathways, but these are positive regulators of both pathways. Nevertheless, MYBs with opposite effects on both pathways have been described in other species before.
Sequence analysis of VvMYB86 revealed the presence of a bHLH interaction motif suggesting that this MYBs might be involved in a MBW complex. Protoplast transfections confirmed that MYB86 is able to activate LAR and to slightly repress ANS and UFGT promoters.


### (20) Anthocyanin activating AtMYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1409383560037892101">TWEET</a>:_** The three anthocyanin biosynthesis regulators PAP1 (MYB75), PAP2 (MYB90), and MYB113 are expressed in different developmental stages of Arabidopsis thaliana leaves #MybMonday (details: https://bit.ly/3jxuQRd)

<a href="https://twitter.com/boas_pucker/status/1409383560037892101">
<img alt="Anthocyanin activating AtMYBs (Tweet #20)" src="https://pbs.twimg.com/media/E47LvU_WEAcx8W8?format=jpg&name=large" width="30%" height="30%">
</a>


There are three anthocyanin activating MYBs in _Arabidopsis thaliana_: MYB75, MYB90, and MYB113. Trans-acting siRNA 4 (TAS4) and microRNA828 (miR828) are repressing the gene expression of these MYBs. [Koo & Poethig, 2021](https://doi.org/10.1186/s13765-020-00584-0) show that the expression of _PAP1_ (_MYB75_), _PAP2_ (_MYB90_), and _MYB113_ change over developmental stages of _A. thaliana_ leaves. TAS4 seems to be most active in young leaves. The anthocyanin content is equal in all leaves during the first 3-4 weeks, but there is a substantially higher anthocyanin concentration in apical leaves in older plants. An additional finding is that _PAP2_ might be responsible for the anthocyanin increase under nutrient-deficient conditions, because its gene expression was increased 1000-fold. Different transcript abundances of _PAP1_ and _PAP2_ could be determined by different affinities of TAS4 for both MYBs.



### (21) MYB111 controls flavonol diversity in sunflowers ###

> **_<a href="https://twitter.com/boas_pucker/status/1411935374494998533">TWEET</a>:_** Cis-regulatory variation in the MYB111 gene contributes to floral pigment diversity of sunflowers. Patterns of UV-absorbing flavonol glycosides are crucial to pollinator attraction and might contribute to temperature regulation #MybMonday (details: https://bit.ly/3jLVZQe)

<a href="https://twitter.com/boas_pucker/status/1411935374494998533">
<img alt="MYB111 controls flavonol diversity in sunflowers (Tweet #21)" src="https://pbs.twimg.com/media/E5fbXVbXMAAtzj9?format=png&name=900x900" width="30%" height="30%">
</a>

[Todesco et al., 2021](https://doi.org/10.1101/2021.06.26.449999) investigate the regulation of the floral pigment diversity of sunflowers. Floral pigmentation is an outstanding example of adapative variation. A pattern of UV absorbing pigments cannot be seen by the human eye, but probably is crucial for the pollinator attraction. A proportion of the sunflower accessions lacks such "nectar guides" due to completely UV-absorbing ligules. This suggest that the pollinator orientation is not one of the flavonol functions in these flowers. GWAS revealed a region on Chr15 as causal for this phenotypic difference between accessions. While this region does not contain a promising candidate gene, MYB111 is located close by. MYB111 appears responsible for the flavonol formation in petals of _A. thaliana_. It seems that MYB12 is not involved in the activation of flavonol accumulation in this tissue type. The explanation is a spatial separation of _MYB111_ and _MYB12_ expression with only _MYB111_ being expressed in these tissues. _MYB111_ shows a particularly high expression in developing ligules. Huge variation in the promoter region of _MYB111_ was observed between sunflower accessions. It appears that "nectar guides" are stronger in cultivated accessions compared to the ancestral state in the wild types, which probably leads to more pollinator visits. UV-absorbing compounds could lead to temperatue increase and make the flower more attractive to pollinators, but the effect is probably neglectable. Flavonols are also involved in drought response by reducing water loss. Since evaporation results in cooling, this could be another mechanism to regulate the flower temperature. 


### (22) Pepper MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1414472089126510593">TWEET</a>:_** A genome-wide analysis of R2R3-MYBs in Capsicum annuum revealed six candidates that might regulate the capsaicin and dihydrocapsaicin biosynthesis #MybMonday (details: https://bit.ly/36m7N3Z)

<a href="https://twitter.com/boas_pucker/status/1414472089126510593">
<img alt="Pepper MYBs (Tweet #22)" src="https://pbs.twimg.com/media/E506B0LWYAM3wfV?format=jpg&name=large" width="30%" height="30%">
</a>

[Wang et al., 2020](https://doi.org/10.3389/fgene.2020.598183) identified 108 R2R3-MYBs in Capsicum annuum. Pepper is well known for its piciness which is due to capsaicin. The authors expected a connection between the MYBs and the capsaicin biosynthesis. Gene structure, protein domain structure, and phylogenetic relationships of the MYBs were summarized in one figure. Individual MYBs were identified as co-expressed with single genes of the capsaicinoid biosynthesis, but it appears that there is no general regulator. CaMYB108 was previously identified as a capsaicin biosynthesis regulator. There might be other/additional transcriptional regulators which were not identified in this study.


### (23) _Morella rubra_ MYBs ###

> **_<a href="https://twitter.com/boas_pucker/status/1417008803967426561">TWEET</a>:_** _Morella rubra_ has 174 MYBs. Overexpression of MrMYB12 causes an increase in flavonol biosynthesis gene expression (_CHS_, _F3H_, _FLS_) and results in lower flux into anthocyanin biosynthesis #MybMonday (details: https://bit.ly/3xLwrae)

<a href="https://twitter.com/boas_pucker/status/1417008803967426561">
<img alt="_Morella rubra_ MYBs (Tweet #23)" src="https://pbs.twimg.com/media/E6Z3piNXIAMiulT?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


[Cao et al., 2021](https://www.frontiersin.org/articles/10.3389/fpls.2021.691384/full) describe the genome-wide investigation of the MYB gene family in Chinese bayberry (_Morella rubra_). _M. rubra_ is a subtropical fruit tree native to China. The 174 MYBs include 122 R3R3-MYBs, 43 1R-MYBs, two 3R-MYBs, one 4R-MYB, and six atypical MYBs. Tandem duplications and larger segmental duplications contributed to the total number of observed MYBs. _M. rubra_ is known for the production of various flavonoids. Therefore, the MrMYBs involved in the flavonoid biosynthesis were investigated in detail. MrMYB12 was identified as a flavonol regulator that can activate the gene expression of _CHS_, _F3H_, and _FLS_. MrMYB39 and MrMYB58a can activate the LAR promoter indicating that they are proanthocyanin regulators.



### (24) _Mimulus_ MYB5a ###

> **_<a href="https://twitter.com/boas_pucker/status/1419530420115365888">TWEET</a>:_** MYB5a is a key regulator of the anthocyanin biosynthesis in the genus Mimulus and contributes to the flower coloration diversity through a reaction-diffusion network #MybMonday (details: https://bit.ly/3iNIAoB)

<a href="https://twitter.com/boas_pucker/status/1419530420115365888">
<img alt=" _Mimulus_ MYB5a (Tweet #24)" src="https://pbs.twimg.com/media/E7LPz2HXIAIXBs3?format=jpg&name=4096x4096" width="30%" height="30%">
</a>


[Zheng et al., 2021](https://doi.org/10.1093/genetics/iyaa036) investigated the coloration of flowers in the genus _Mimulus_. _MYB5a_/_NEGAN_ was previously identified as a crucial factor determining the visual diversity. _NEGAN_/_MYB5_ is part of a reaction-diffusion system in _M. lewisii_ and _M. guttatus_ that leads to spots of red anthocyanin pigmentation in the nectar guide region [Ding et al., 2020](https://pubmed.ncbi.nlm.nih.gov/32155414/). Genetic mapping revealed _MYB4_ and _MYB5a_ as candidates in a QTL for anthocyanin pigmentation. Subsequent gene expression analysis showed that _MYB5a_ is the most likely candidate due to a specific expression in the petals. The yellow flowering _M. luteus luteus_ and the anthocyanin-pigmented _M. luteus variegatus_ differ by an InDel in the third exon of _MYB5a_ which excludes the subgroup 6 motif from the _M. luteus variegatus_ MYB. However, the third and the fourth exon are very similar (duplication?) suggesting that the fourth exon contributes the subgroup 6 motif. It appears that there are two different splice variants of the _MYB5_: exon1-2-4 in the anthocyanin-pigmented tissues and exon1-2-3 in other tissues. RNAi knockdown of _MYB5a_ was used to demonstrate the involvement in anthocyanin biosynthesis regulation. _MYB5a_ is an activator of its own inhibitor _RTO_.


### (25) R2R3-MYBs in tea plant ###

> **_<a href="https://twitter.com/boas_pucker/status/1422070909108637696">TWEET</a>:_** Genome-wide investigation of R2R3-MYBs in Camellia sinensis revealed candidates for the regulation of galloylated catechin biosynthesis #MybMonday (details: https://bit.ly/3ykOWTd)

<a href="https://twitter.com/boas_pucker/status/1422070909108637696">
<img alt="R2R3-MYBs in tea plant (Tweet #25)" src="https://pbs.twimg.com/media/E7wycUrXEAcr9Zy?format=jpg&name=medium" width="30%" height="30%">
</a>

[Li et al., 2021](https://doi.org/10.1101/2021.07.21.453189) analazyed the R2R3-MYBs in _Camelia sinensis_ (tea plant) to find the regulator of the galloylated catechin biosynthesis. Catechins accound for 70% of the polyphenols in _C. sinensis_ and are the major health promoting factor. Most abundant are galloylated catechins (ECG and EGCG). A total of 118 R2R3-MYBs were discovered in _C. sinensis_ and presented in a phylogenetic tree with the _A. thaliana_ MYBs (Dubos et al., 2010). No tea plant MYB falls in the S12 group comprized of AtMYB28, AtMYB29, AtMYB76, AtMYB34, and AtMYB51. These are Brassicaceae-specific MYBs that regulate glucosinolate biosynthesis (Gigolashvili et al., 2007; Matus et al., 2008). Three MYB subgroups without _A. thaliana_ members were identified. The authors propose that these evolved in tea plant, but the presence of _Vitis vinifera_ MYBs in these clades suggest that they have been lost in _A. thaliana_ instead. The authors decided to focus on subgroup 5 due to expression in apical buds and young leaves where high catechin concentrations were detected. _CsMYB30_, _CsMYB34_, _CsMYB37_ and _CsMYB42_ revealed a strong co-expression with the catechin biosynthesis genes (_CsMYB32_ instead of _CsMYB42_ was observed in a validation experiment). 


### (26) Mutation in R2R3-MYB _GmTDF1-1_ triggers male sterility in soybean ###

> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** Disruption of the R2R3-MYB GmTDF1-1 by a point mutation triggers male sterility in soybean. The very similar GmTDF1-2 is too weakly expressed to compensate this loss of GmTDF1-1 activity #MybMonday (details: https://bit.ly/3xw7ej1)



[Yu et al, 2021](https://link.springer.com/article/10.1007/s00122-021-03920-0) report that the R2R3-MYB transcription factor GmTDF1-1 is crucial for the anther development in soybean. A loss of this gene function by a point mutation resulted in male sterility. There is a very similar _TDF1-2_ gene, but insufficient expression prevents a compensation of the _TDF1-1_ function loss. Male sterility is an important trait in soybean, because it facilitates hybrid breeding.



### (27) Evolution of the 3R-MYB gene family ###

> **_<a href="https://github.com/bpucker/MybMonday">TWEET</a>:_** 3R-MYBs (R1R2R3-MYBs) are involved in the cell cycle and in abiotic stress response. It might be an adaptation to a sessile lifestyle. #MybMonday (details: )



[Feng et al., 2017](https://pubmed.ncbi.nlm.nih.gov/28444194/) investigated the evolution of the 3R-MYB gene family. Since most plant MYBs belong to the R2R3-MYB gene family, the other families are often neglected. It is important to note that this classification based on the presence of repeats might not reflect the phylogeny of MYBs. When investigating the function of MYBs, it should be better to focus on their phylogeny instead on the presence of repeats. 3R-MYBs are divided into three groups: A, B, and C. Synteny analysis indicates that the firsts 3R-MYB gene was amplified through segmental duplications in Amborella. Gene duplications within these lineages after the separation of monocots and dicots are the important factor that explains the large diversity seen today. The general gene structure of 3R-MYBs changed during evolution through the acquisition of additional introns.  What could have been the evolutionary advantage? There are many cases of alternative splicing, but only a small proportion seems to change the final protein. Most events appear to be unproductive splicing or are restricted to the UTR. These events might be an additional layer of gene expression regulation. Alternative splicing in the CDS could also cause the loss of a repressor motif leading to a hyperactive protein. Helix2 and helix3 encoded by a MYB repeat are binding DNA and thus more conserved. Especially helix3 does not show much variation, because this helix forms the direct contact to the DNA.



## Contact ##
Do you have a recent publication about MYBs in plants that could be featured here? Please get in touch: [Dr. Boas Pucker (email)](mailto:bpucker@cebitec.uni-bielefeld.de?subject=[GitHub]MYB_Monday)


## References ##

<details>
<summary>(click to expand)</summary>
<p>
Pucker B., Pandey A., Weisshaar B. and Stracke R. (2020). The R2R3-MYB gene family in banana (Musa acuminata): genome-wide identification, classification and expression patterns. PLOS ONE 15(10): e0239275. doi:[10.1371/journal.pone.0239275](https://doi.org/10.1371/journal.pone.0239275).

Thiedig K, Weisshaar B, Stracke R (2021). Functional and evolutionary analysis of the Arabidopsis 4R-MYB protein SNAPc4 as part of the SNAP complex. Plant Physiology. doi:[10.1093/plphys/kiaa067](https://pub.uni-bielefeld.de/record/2949074).

Dubos C, Stracke R, Grotewold E, Weisshaar B, Martin C, Lepiniec L (2010). MYB transcription factors in Arabidopsis. Trends in Plant Science 15(10): 573-581. doi:[10.1016/j.tplants.2010.06.005 ](https://pubmed.ncbi.nlm.nih.gov/20674465/).

Baudry A, Heim MA, Dubreucq B, Caboche M, Weisshaar B, Lepiniec L (2004). TT2, TT8, and TTG1 synergistically specify the expression of BANYULS and proanthocyanidin biosynthesis in Arabidopsis thaliana. doi:[10.1111/j.1365-313X.2004.02138.x](https://pubmed.ncbi.nlm.nih.gov/15255866/).

Nozomi Haga, Kiichi Kato, Masatake Murase, Satoshi Araki, Minoru Kubo, Taku Demura, Kaoru Suzuki, Isabel Müller, Ute Voß, Gerd Jürgens, Masaki Ito (2007). R1R2R3-Myb proteins positively regulate cytokinesis through activation of KNOLLE transcription in Arabidopsis thaliana. Development 2007 134: 1101-1110; doi: [10.1242/dev.02801](https://dev.biologists.org/content/134/6/1101)
</p>
</details>
